<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> EG1 </td></tr><tr><th>Gene symbol synonym(s)</th><td> eg1 GY1 </td></tr><tr><th>CGSNL Gene Name</th><td> EXTRA GLUME 1 </td></tr><tr><th>Gene name synonym(s)</th><td> extra glume-1 extra glume 1 extra glume1 gaoyao1 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Tolerance and resistance - Stress tolerance Reproductive organ - Spikelet, flower, glume, awn </td></tr><tr><th>Gene Ontology</th><td> GO:0008970 - phospholipase A1 activity GO:0009408 - response to heat GO:0009507 - chloroplast GO:0009695 - jasmonic acid biosynthetic process GO:0009723 - response to ethylene stimulus GO:0009873 - ethylene mediated signaling pathway GO:0016042 - lipid catabolic process GO:0030154 - cell differentiation GO:0048437 - floral organ development </td></tr><tr><th>Trait Ontology</th><td> TO:0000079 - lemma and palea anatomy and morphology trait TO:0002668 - jasmonic acid content TO:0000657 - spikelet anatomy and morphology trait TO:0006022 - floral organ development trait TO:0000544 - mesocotyl length TO:0001007 - coleoptile length TO:0000173 - ethylene sensitivity </td></tr><tr><th>Plant Ontology</th><td> PO:0009010 - seed PO:0009082 - spikelet floret PO:0009039 - glume </td></tr><tr><th>RAP ID</th><td> Os01g0900400 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> image Id ( 6722 ) </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0900400</td></tr><tr><th>name</th><td>EXTRA GLUME 1</td></tr><tr><th>description</th><td>Putative triacylglycerol (TAG) lipase, Phospholipase A1, Specification of empty-glume identity, Regulation of spikelet development, Endogenous jasmonic acid (JA) biosynthesis (Os01t0900400-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9851</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>39177169</td></tr><tr><th>end</th><td>39178676</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>EG1</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000AA3ED</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q8S1D9</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g67430.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAF07019.2</li><li>BAB89948.1</li><li>BAS75742.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK242904</li></ul></td></tr><tr><td>RAP-DB</td><td><ul><li>EG1, eg1</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>eg1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR002921</td><td>Fungal_lipase-like</td><td>Fungal lipase-like domain</td></tr><tr><td>IPR029058</td><td>AB_hydrolase</td><td>Alpha/Beta hydrolase fold</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0016042</td><td>lipid catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of lipids, compounds soluble in an organic solvent but not, or sparingly, in an aqueous solvent.</td><td>IEA</td></tr><tr><td>GO:0052740</td><td>1-acyl-2-lysophosphatidylserine acylhydrolase activity</td><td>molecular_function</td><td>Catalysis of the reaction: 1-acyl-2-lysophosphatidylserine + H2O = sn-glycerol-phosphoserine + a carboxylate.</td><td>IEA</td></tr><tr><td>GO:0008970</td><td>phosphatidylcholine 1-acylhydrolase activity</td><td>molecular_function</td><td>Catalysis of the reaction: phosphatidylcholine + H2O = 2-acylglycerophosphocholine + a carboxylate.</td><td>IEA</td></tr><tr><td>GO:0009507</td><td>chloroplast</td><td>cellular_component</td><td>A chlorophyll-containing plastid with thylakoids organized into grana and frets, or stroma thylakoids, and embedded in a stroma.</td><td>IDA</td></tr><tr><td>GO:0052739</td><td>phosphatidylserine 1-acylhydrolase activity</td><td>molecular_function</td><td>Catalysis of the reaction: phosphatidylserine + H2O = 2-acyl-sn-glycero-3-phosphoserine + fatty acid.</td><td>IEA</td></tr><tr><td>GO:0048449</td><td>floral organ formation</td><td>biological_process</td><td>The process that gives rise to floral organs. This process pertains to the initial formation of a structure from unspecified parts.</td><td>IDA</td></tr><tr><td>GO:0005739</td><td>mitochondrion</td><td>cellular_component</td><td>A semiautonomous, self replicating organelle that occurs in varying numbers, shapes, and sizes in the cytoplasm of virtually all eukaryotic cells. It is notably the site of tissue respiration.</td><td>IDA</td></tr><tr><td>GO:0010582</td><td>floral meristem determinacy</td><td>biological_process</td><td>The process in which a floral meristem becomes determinate (i.e. ceases to produce lateral organs and may or may not terminally differentiate).</td><td>IMP</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G43990</li><li>BGIOSGA004990</li><li>ONIVA01G45710</li><li>ORGLA01G0346100</li><li>OMERI01G37640</li><li>OPUNC01G39420</li><li>HORVU3Hr1G089140</li><li>TraesCS3B01G427100</li><li>TraesCS3A01G395300</li><li>BRADI_2g57760v3</li><li>LPERR01G35220</li><li>TraesCS3D01G388900</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0900700</li><li>Os11g0299300</li><li>Os05g0390000</li><li>Os02g0653900</li><li>Os08g0143600</li><li>Os10g0562200</li><li>Os11g0146600</li><li>Os05g0574100</li><li>Os05g0574000</li><li>Os01g0651800</li><li>Os01g0710700</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>Bo5g006970</li><li>Bo01021s020</li><li>fgenesh2_kg.4__1154__AT2G31690.1</li><li>fgenesh2_kg.1__541__AT1G05800.1</li><li>MANES_11G113300</li><li>MANES_04G053400</li><li>Solyc09g098460.1</li><li>B456_006G228000</li><li>B456_005G017700</li><li>GSBRNA2T00132163001</li><li>PGSC0003DMG400042356</li><li>GSBRNA2T00056879001</li><li>Solyc12g098730.1</li><li>HannXRQ_Chr05g0152761</li><li>GSBRNA2T00142055001</li><li>MANES_11G113200</li><li>PGSC0003DMG400021618</li><li>B456_011G144700</li><li>GLYMA_18G299300</li><li>POPTR_002G231600v3</li><li>Bra015454</li><li>AT2G31690</li><li>HannXRQ_Chr05g0152751</li><li>TanjilG_08862</li><li>Bra022862</li><li>Bo3g026380</li><li>AT1G05800</li><li>GSBRNA2T00003208001</li><li>POPTR_014G150700v3</li><li>TanjilG_27650</li><li>GLYMA_08G362700</li><li>GSBRNA2T00031959001</li><li>PHYPA_006488</li><li>PHYPA_006316</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'EG1|GY1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0900400'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g67430'}</td></tr><tr><th>Keyword</th><td>{'JA biosynthesis', 'spikelet', 'floral meristem', 'jasmonic', 'ethylene', 'jasmonic acid', 'JA', 'growth', 'map-based cloning', 'cell elongation', 'seedlings', ' ja ', 'meristem', 'temperature', 'inflorescence', 'floral'}</td></tr><tr><th>Title</th><td>{'Jasmonic acid regulates spikelet development in rice', 'A High Temperature-Dependent Mitochondrial Lipase EXTRA GLUME1 Promotes Floral Phenotypic Robustness against Temperature Fluctuation in Rice (Oryza sativa L.).', 'A putative lipase gene EXTRA GLUME1 regulates both empty-glume fate and spikelet development in rice', 'Ethylene-Inhibited Jasmonic Acid Biosynthesis Promotes Mesocotyl/Coleoptile Elongation of Etiolated Rice Seedlings.'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0900400</td></tr><tr><th>Description</th><td>Putative triacylglycerol (TAG) lipase, Phospholipase A1, Specification of empty-glume identity, Regulation of spikelet development, Endogenous jasmonic acid (JA) biosynthesis (Os01t0900400-01)</td></tr><tr><th>Position</th><td>chr01:39177169..39178676</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>EG1, eg1</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>EXTRA GLUME 1, extra glume-1, extra glume 1, extra glume1</td></tr><tr><th>CGSNL Gene Symbol</th><td>EG1</td></tr><tr><th>CGSNL Gene Name</th><td>EXTRA GLUME 1</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>eg1, GY1</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>extra glume-1, extra glume 1, extra glume1, gaoyao1</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'EG1|GY1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0900400'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g67430'}</td></tr></table></td></tr></table>