<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> GLYII-1 </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsGLYII1 GLYII1 OsETHE1 ETHE1 OsGLYII-1 </td></tr><tr><th>CGSNL Gene Name</th><td> GLYOXALASE II-1 </td></tr><tr><th>Gene name synonym(s)</th><td> glyoxalase II-1 ETHYLMALONIC ENCEPHALOPATHY PROTEIN 1 ETHE1-like protein ethylmalonic encephalopathy-1 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> </td></tr><tr><th>Trait Ontology</th><td> TO:0000615 - abscisic acid sensitivity TO:0002657 - oxidative stress TO:0000172 - jasmonic acid sensitivity TO:0000394 - drought related trait TO:0000429 - salt sensitivity TO:0000432 - temperature response trait </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0667200 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0667200</td></tr><tr><th>name</th><td>GLYOXALASE II-1</td></tr><tr><th>description</th><td>Mitochondrial sulfur dioxygenase, Abiotic stress response (Os01t0667200-01);Similar to Glyoxalase II. (Os01t0667200-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>6569</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>27274750</td></tr><tr><th>end</th><td>27278556</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>GLYII-1</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI000048E021</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q5QLQ5</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g47690.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015651425.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK104749</li><li>AK104104</li><li>AK067930</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS73586.1</li><li>BAF05721.1</li><li>BAD73645.1</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015795939.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326319</li></ul></td></tr><tr><td>RAP-DB</td><td><ul><li>GLYII-1, OsGLYII1, GLYII1, OsETHE1, ETHE1, OsGLYII-1</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsGLYII1, GLYII1, OsETHE1, ETHE1, OsGLYII-1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR001279</td><td>Metallo-B-lactamas</td><td>Metallo-beta-lactamase</td></tr><tr><td>IPR036866</td><td>RibonucZ/Hydroxyglut_hydro</td><td>Ribonuclease Z/Hydroxyacylglutathione hydrolase-like</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0016788</td><td>hydrolase activity, acting on ester bonds</td><td>molecular_function</td><td>Catalysis of the hydrolysis of any ester bond.</td><td>IEA</td></tr><tr><td>GO:0050313</td><td>sulfur dioxygenase activity</td><td>molecular_function</td><td>Catalysis of the reaction: sulfur + O2 + H2O = sulfite.</td><td>IEA</td></tr><tr><td>GO:0009651</td><td>response to salt stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating an increase or decrease in the concentration of salt (particularly but not exclusively sodium and chloride ions) in the environment.</td><td>IEA</td></tr><tr><td>GO:0009793</td><td>embryo development ending in seed dormancy</td><td>biological_process</td><td>The process whose specific outcome is the progression of the embryo over time, from zygote formation to the end of seed dormancy. An example of this process is found in Arabidopsis thaliana.</td><td>IEA</td></tr><tr><td>GO:0009960</td><td>endosperm development</td><td>biological_process</td><td>The process whose specific outcome is the progression of the endosperm over time, from its formation to the mature structure. The endosperm is formed during fertilization and provides nutrients to the developing embryo.</td><td>IEA</td></tr><tr><td>GO:0005739</td><td>mitochondrion</td><td>cellular_component</td><td>A semiautonomous, self replicating organelle that occurs in varying numbers, shapes, and sizes in the cytoplasm of virtually all eukaryotic cells. It is notably the site of tissue respiration.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>BGIOSGA001060</li><li>ORUFI01G28710</li><li>ORGLA01G0211200</li><li>OBART01G25800</li><li>KN540422.1_FG004</li><li>OGLUM01G29640</li><li>OPUNC01G25710</li><li>OB01G34980</li><li>LPERR01G22060</li><li>TraesCS3A01G247300</li><li>TRIUR3_22119</li><li>BRADI_2g46040v3</li><li>F775_26821</li><li>TraesCS3D01G244100</li><li>TraesCS3B01G272000</li><li>SORBI_3003G249900</li><li>SETIT_002444mg</li><li>Zm00001d043941</li><li>Dr13159</li><li>Al_scaffold_0001_4800</li><li>MANES_02G089400</li><li>MTR_2g072190</li><li>PHAVU_005G062700g</li><li>BVRB_4g094020</li><li>HannXRQ_Chr10g0305671</li><li>AT1G53580</li><li>Csa_4G621190</li><li>TanjilG_31908</li><li>PRUPE_3G177200</li><li>LR48_Vigan05g026000</li><li>DCAR_019628</li><li>AMTR_s00014p00246150</li><li>Gasu_08440</li><li>OSTLU_38033</li><li>WBGene00007886</li><li>FBgn0050022</li><li>ENSG00000105755</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os03g0332400</li><li>Os09g0516600</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsGLYII1|OsETHE1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0667200'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g47690'}</td></tr><tr><th>Keyword</th><td>{'stress', 'calcium', 'stress response', 'root'}</td></tr><tr><th>Title</th><td>{'Stress response of OsETHE1 is altered in response to light and dark conditions.', 'Expression of abiotic stress inducible ETHE1-like protein from rice is higher in roots and is regulated by calcium'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0667200</td></tr><tr><th>Description</th><td>Mitochondrial sulfur dioxygenase, Abiotic stress response (Os01t0667200-01);Similar to Glyoxalase II. (Os01t0667200-02)</td></tr><tr><th>Position</th><td>chr01:27274750..27278556</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>GLYII-1, OsGLYII1, GLYII1, OsETHE1, ETHE1, OsGLYII-1</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>GLYOXALASE II-1, glyoxalase II-1, ETHYLMALONIC ENCEPHALOPATHY PROTEIN 1, ETHE1-like protein, ethylmalonic encephalopathy-1</td></tr><tr><th>CGSNL Gene Symbol</th><td>GLYII-1</td></tr><tr><th>CGSNL Gene Name</th><td>GLYOXALASE II-1</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsGLYII1, GLYII1, OsETHE1, ETHE1, OsGLYII-1</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>glyoxalase II-1, ETHYLMALONIC ENCEPHALOPATHY PROTEIN 1, ETHE1-like protein, ethylmalonic encephalopathy-1</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsGLYII1|OsETHE1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0667200'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g47690'}</td></tr></table></td></tr></table>