<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsBT BT </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> BT1/BT2 ortholog </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Other </td></tr><tr><th>Gene Ontology</th><td> GO:0003712 - transcription cofactor activity GO:0004402 - histone acetyltransferase activity GO:0005634 - nucleus GO:0006355 - regulation of transcription, DNA-dependent GO:0008270 - zinc ion binding GO:0015706 - nitrate transport </td></tr><tr><th>Trait Ontology</th><td> TO:0000329 - tillering ability </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0908200 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0908200</td></tr><tr><th>name</th><td>OsBT</td></tr><tr><th>description</th><td>Member of the Bric-a-Brac/Tramtrack/Broad (BTB) family, BT1/BT2 ortholog, Negative regulation of nitrate uptake and nitrogen use efficiency (Os01t0908200-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9946</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>39544235</td></tr><tr><th>end</th><td>39548599</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>RAP-DB</td><td><ul><li>OsBT</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000A2577</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q8L3R7</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g68020.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015614446.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>EAZ14554.1</li><li>BAB92604.1</li><li>BAF07055.1</li><li>BAS75808.1</li><li>BAB92571.1</li><li>BAD82754.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK067269</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015758960.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4324985</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsBT, BT</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR000197</td><td>Znf_TAZ</td><td>Zinc finger, TAZ-type</td></tr><tr><td>IPR000210</td><td>BTB/POZ_dom</td><td>BTB/POZ domain</td></tr><tr><td>IPR011333</td><td>SKP1/BTB/POZ_sf</td><td>SKP1/BTB/POZ domain superfamily</td></tr><tr><td>IPR034089</td><td>SPOP_C</td><td>SPOP, C-terminal BACK-like domain</td></tr><tr><td>IPR035898</td><td>TAZ_dom_sf</td><td>TAZ domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IBA</td></tr><tr><td>GO:0019005</td><td>SCF ubiquitin ligase complex</td><td>cellular_component</td><td>A ubiquitin ligase complex in which a cullin from the Cul1 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a Skp1 adaptor and an F-box protein. SCF complexes are involved in targeting proteins for degradation by the proteasome. The best characterized complexes are those from yeast and mammals (with core subunits named Cdc53/Cul1, Rbx1/Hrt1/Roc1).</td><td>IBA</td></tr><tr><td>GO:0008270</td><td>zinc ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with zinc (Zn) ions.</td><td>IEA</td></tr><tr><td>GO:0030162</td><td>regulation of proteolysis</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein.</td><td>IBA</td></tr><tr><td>GO:0043161</td><td>proteasome-mediated ubiquitin-dependent protein catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of ubiquitin, and mediated by the proteasome.</td><td>IBA</td></tr><tr><td>GO:0004402</td><td>histone acetyltransferase activity</td><td>molecular_function</td><td>Catalysis of the reaction: acetyl-CoA + histone = CoA + acetyl-histone.</td><td>IEA</td></tr><tr><td>GO:0031625</td><td>ubiquitin protein ligase binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a ubiquitin protein ligase enzyme, any of the E3 proteins.</td><td>IBA</td></tr><tr><td>GO:0006355</td><td>regulation of transcription, DNA-templated</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of cellular DNA-templated transcription.</td><td>IEA</td></tr><tr><td>GO:0003712</td><td>transcription coregulator activity</td><td>molecular_function</td><td>A protein or a member of a complex that interacts specifically and non-covalently with a DNA-binding transcription factor to either activate or repress the transcription of specific genes. Coregulators often act by altering chromatin structure and modifications. For example, one class of transcription coregulators modifies chromatin structure through covalent modification of histones. A second ATP-dependent class modifies the conformation of chromatin. Another type of coregulator activity is the bridging of a DNA-binding transcription factor to the basal transcription machinery. The Mediator complex, which bridges transcription factors and RNA polymerase, is also a transcription coregulator (specifically, a coactivator).</td><td>IEA</td></tr><tr><td>GO:0005737</td><td>cytoplasm</td><td>cellular_component</td><td>All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.</td><td>IBA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ONIVA01G46180</li><li>ORGLA01G0350600</li><li>ORUFI01G44480</li><li>BGIOSGA005014</li><li>OBART01G41150</li><li>KN539003.1_FG016</li><li>OMERI01G38160</li><li>OPUNC01G39960</li><li>OB01G49860</li><li>LPERR01G35690</li><li>TRIUR3_25289</li><li>BRADI_2g58152v3</li><li>HORVU3Hr1G092090</li><li>Zm00001d042443</li><li>SETIT_002070mg</li><li>SORBI_3003G395400</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os04g0482300</li><li>Os02g0596700</li><li>Os01g0893400</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>LR48_Vigan01g228500</li><li>LR48_Vigan04g071200</li><li>HannXRQ_Chr01g0027301</li><li>AT3G48360</li><li>PGSC0003DMG400015229</li><li>A4A49_00302</li><li>GLYMA_09G098400</li><li>GSBRNA2T00141561001</li><li>AT5G63160</li><li>GLYMA_04G190900</li><li>POPTR_015G087700v3</li><li>GLYMA_05G0439001</li><li>A4A49_00301</li><li>MANES_06G113100</li><li>GSBRNA2T00006902001</li><li>GLYMA_17G126400</li><li>GSBRNA2T00088986001</li><li>GSBRNA2T00046095001</li><li>DCAR_016431</li><li>Solyc06g071830.2</li><li>Bo3g104690</li><li>B456_001G062300</li><li>Csa_1G049460</li><li>GSBRNA2T00068129001</li><li>HannXRQ_Chr04g0100771</li><li>MANES_14G056500</li><li>DCAR_021599</li><li>Solyc06g071820.2</li><li>Bra029235</li><li>Bo2g165450</li><li>Bra018065</li><li>Csa_1G032450</li><li>GSBRNA2T00073117001</li><li>MTR_4g104140</li><li>MTR_3g075270</li><li>PGSC0003DMG402026984</li><li>POPTR_012G091200v3</li><li>fgenesh1_pm.C_scaffold_5000980</li><li>DCAR_026199</li><li>Bo8g076850</li><li>GSBRNA2T00082655001</li><li>PHAVU_003G207300g</li><li>A4A49_42024</li><li>HannXRQ_Chr02g0035281</li><li>B456_008G204000</li><li>Solyc03g111710.2</li><li>PHAVU_009G170600g</li><li>scaffold_803004.1</li><li>HannXRQ_Chr09g0272511</li><li>A4A49_17125</li><li>PGSC0003DMG401026984</li><li>Bra038612</li><li>POPTR_012G091400v3</li><li>SELMODRAFT_13821</li><li>SELMODRAFT_13820</li><li>PHYPA_020045</li><li>PHYPA_029222</li><li>PHYPA_012980</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsBT'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0908200'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g68020'}</td></tr><tr><th>Keyword</th><td>{'nitrogen'}</td></tr><tr><th>Title</th><td>{'Members of BTB gene family regulate negatively nitrate uptake and nitrogen use efficiency in Arabidopsis thaliana and Oryza sativa.'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0908301</td></tr><tr><th>Description</th><td>Non-protein coding transcript. (Os01t0908301-01)</td></tr><tr><th>Position</th><td>chr01:39544770..39545927</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td></td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsBT'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0908200'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g68020'}</td></tr></table></td></tr></table>