<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> FTSH9 </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsFtsH9 FtsH9 </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0009409 - response to cold GO:0005739 - mitochondrion GO:0031966 - mitochondrial membrane GO:0009941 - chloroplast envelope GO:0010304 - PSII associated light-harvesting complex II catabolic process GO:0006508 - proteolysis GO:0046872 - metal ion binding GO:0016021 - integral to membrane GO:0004176 - ATP-dependent peptidase activity GO:0004222 - metalloendopeptidase activity GO:0005524 - ATP binding GO:0008237 - metallopeptidase activity GO:0009535 - chloroplast thylakoid membrane </td></tr><tr><th>Trait Ontology</th><td> TO:0000303 - cold tolerance </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0618800 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0618800</td></tr><tr><th>name</th><td>FTSH9</td></tr><tr><th>description</th><td>Similar to ATP-dependent zinc metalloprotease FTSH 9, chloroplastic/mitochondrial. (Os01t0618800-01);Similar to Cell division protease ftsH homolog 9, chloroplastic/mitochondrial. (Os01t0618800-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>5923</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>24616103</td></tr><tr><th>end</th><td>24618581</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>FTSH9</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0000DD8C05</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>A0A0N7KDC0</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS73194.1</li><li>BAS73193.1</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsFtsH9, FtsH9</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR000642</td><td>Peptidase_M41</td><td>Peptidase M41</td></tr><tr><td>IPR003959</td><td>ATPase_AAA_core</td><td>ATPase, AAA-type, core</td></tr><tr><td>IPR003960</td><td>ATPase_AAA_CS</td><td>ATPase, AAA-type, conserved site</td></tr><tr><td>IPR027417</td><td>P-loop_NTPase</td><td>P-loop containing nucleoside triphosphate hydrolase</td></tr><tr><td>IPR037219</td><td>Peptidase_M41-like</td><td>Peptidase M41-like</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0005524</td><td>ATP binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator.</td><td>IEA</td></tr><tr><td>GO:0004222</td><td>metalloendopeptidase activity</td><td>molecular_function</td><td>Catalysis of the hydrolysis of internal, alpha-peptide bonds in a polypeptide chain by a mechanism in which water acts as a nucleophile, one or two metal ions hold the water molecule in place, and charged amino acid side chains are ligands for the metal ions.</td><td>IEA</td></tr><tr><td>GO:0004176</td><td>ATP-dependent peptidase activity</td><td>molecular_function</td><td>Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive the hydrolysis of peptide bonds.</td><td>IBA</td></tr><tr><td>GO:0046872</td><td>metal ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any metal ion.</td><td>IEA</td></tr><tr><td>GO:0010304</td><td>PSII associated light-harvesting complex II catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of one or more components of the light-harvesting complex of photosystem II.</td><td>IBA</td></tr><tr><td>GO:0009535</td><td>chloroplast thylakoid membrane</td><td>cellular_component</td><td>The pigmented membrane of a chloroplast thylakoid. An example of this component is found in Arabidopsis thaliana.</td><td>IEA</td></tr><tr><td>GO:0009941</td><td>chloroplast envelope</td><td>cellular_component</td><td>The double lipid bilayer enclosing the chloroplast and separating its contents from the rest of the cytoplasm; includes the intermembrane space.</td><td>IBA</td></tr><tr><td>GO:0016021</td><td>integral component of membrane</td><td>cellular_component</td><td>The component of a membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane.</td><td>IEA</td></tr><tr><td>GO:0031966</td><td>mitochondrial membrane</td><td>cellular_component</td><td>Either of the lipid bilayers that surround the mitochondrion and form the mitochondrial envelope.</td><td>IEA</td></tr><tr><td>GO:0003723</td><td>RNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with an RNA molecule or a portion thereof.</td><td>IEA</td></tr><tr><td>GO:0009408</td><td>response to heat</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a heat stimulus, a temperature stimulus above the optimal temperature for that organism.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>KN539181.1_FG003</li><li>OBART01G22910</li><li>ORGLA01G0184400</li><li>BGIOSGA001195</li><li>ORUFI01G25720</li><li>OMERI01G21010</li><li>OGLUM01G26660</li><li>OPUNC01G23110</li><li>OB01G32430</li><li>LPERR01G19690</li><li>BRADI_2g43740v3</li><li>TRIUR3_15527</li><li>TraesCS3A01G222300</li><li>TraesCS3D01G230100</li><li>TraesCS3B01G252900</li><li>SETIT_000384mg</li><li>SORBI_3003G222900</li><li>Dr09257</li><li>Csa_5G139560</li><li>PRUPE_5G026500</li><li>MTR_1g028390</li><li>TCM_015358</li><li>BVRB_3g052420</li><li>Tp57577_TGAC_v2_gene33321</li><li>CCACVL1_19749</li><li>AT5G53170</li><li>VIT_16s0050g02630</li><li>PGSC0003DMG400009105</li><li>TanjilG_15648</li><li>fgenesh2_kg.8__1205__AT5G53170.1</li><li>Solyc03g007330.2</li><li>DCAR_010617</li><li>Bra022635</li><li>AMTR_s00107p00109770</li><li>PHYPA_024830</li><li>CHLRE_14g625625v5</li><li>OSTLU_38163</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0574400</li><li>Os01g0574500</li><li>Os01g0842600</li><li>Os05g0458400</li><li>Os04g0466100</li><li>Os02g0649700</li><li>Os06g0725900</li><li>Os06g0229000</li><li>Os06g0229066</li><li>Os06g0669400</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0618800</td></tr><tr><th>Description</th><td>Similar to ATP-dependent zinc metalloprotease FTSH 9, chloroplastic/mitochondrial. (Os01t0618800-01);Similar to Cell division protease ftsH homolog 9, chloroplastic/mitochondrial. (Os01t0618800-02)</td></tr><tr><th>Position</th><td>chr01:24616103..24618581</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>FTSH9</td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsFtsH9, FtsH9</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr></table>