<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> ATG7 </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsATG7 Atg7 </td></tr><tr><th>CGSNL Gene Name</th><td> AUTOPHAGY ASSOCIATED GENE 7 </td></tr><tr><th>Gene name synonym(s)</th><td> autophagy 7 OsAuTophaGy7 autophagy-related 7 AUTOPHAGY RELATED7 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Reproductive organ - Pollination, fertilization, fertility - Male sterility </td></tr><tr><th>Gene Ontology</th><td> GO:0000166 - nucleotide binding GO:0003824 - catalytic activity GO:0005737 - cytoplasm GO:0006629 - lipid metabolic process GO:0006914 - autophagy GO:0009846 - pollen germination GO:0009901 - anther dehiscence GO:0010150 - leaf senescence GO:0010152 - pollen maturation GO:0046470 - phosphatidylcholine metabolic process GO:0048653 - anther development </td></tr><tr><th>Trait Ontology</th><td> TO:0000106 - male sterility type TO:0000249 - leaf senescence TO:0002637 - leaf size TO:0002675 - gibberellic acid content TO:0002660 - cytokinin content </td></tr><tr><th>Plant Ontology</th><td> PO:0001004 - anther development stage PO:0001035 - G anther dehiscence stage PO:0001054 - 4 leaf senescence stage </td></tr><tr><th>RAP ID</th><td> Os01g0614900 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0614900</td></tr><tr><th>name</th><td>AUTOPHAGY ASSOCIATED GENE 7</td></tr><tr><th>description</th><td>NAD(P)-binding domain containing protein. (Os01t0614900-01);Similar to autophagy-related 7. (Os01t0614900-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>5856</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>24380637</td></tr><tr><th>end</th><td>24386666</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>ATG7</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00004504D9</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q5ZDX5</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g42850.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015615773.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS73152.1</li><li>BAF05483.1</li><li>BAD61236.1</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015760287.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326644</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsATG7, Atg7</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR000594</td><td>ThiF_NAD_FAD-bd</td><td>THIF-type NAD/FAD binding fold</td></tr><tr><td>IPR006285</td><td>Atg7</td><td>Ubiquitin-like modifier-activating enzyme Atg7</td></tr><tr><td>IPR032197</td><td>Atg7_N</td><td>Ubiquitin-like modifier-activating enzyme Atg7, N-terminal</td></tr><tr><td>IPR035985</td><td>Ubiquitin-activating_enz</td><td>Ubiquitin-activating enzyme</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>16040</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0000422</td><td>autophagy of mitochondrion</td><td>biological_process</td><td>The autophagic process in which mitochondria are delivered to the vacuole and degraded in response to changing cellular conditions.</td><td>IBA</td></tr><tr><td>GO:0019779</td><td>Atg8 activating enzyme activity</td><td>molecular_function</td><td>Catalysis of the activation of the small ubiquitin-related modifier APG8, through the formation of an ATP-dependent high-energy thiolester bond.</td><td>IBA</td></tr><tr><td>GO:0006995</td><td>cellular response to nitrogen starvation</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of deprivation of nitrogen.</td><td>IBA</td></tr><tr><td>GO:0006501</td><td>C-terminal protein lipidation</td><td>biological_process</td><td>The covalent attachment of a lipid group to the carboxy terminus of a protein.</td><td>IBA</td></tr><tr><td>GO:0019778</td><td>Atg12 activating enzyme activity</td><td>molecular_function</td><td>Catalysis of the activation of the small ubiquitin-related modifier APG12, through the formation of an ATP-dependent high-energy thiolester bond.</td><td>IBA</td></tr><tr><td>GO:0000407</td><td>phagophore assembly site</td><td>cellular_component</td><td>Punctate structures proximal to the endoplasmic reticulum which are the sites where the Atg machinery assembles upon autophagy induction.</td><td>IBA</td></tr><tr><td>GO:0044805</td><td>late nucleophagy</td><td>biological_process</td><td>A type of nucleophagy, distinct from piecemeal microautophagy of the nucleus (PNM) where the nuclear material is delivered to the vacuole/lysosome for breakdown and recycling later than observed for PNM.</td><td>IBA</td></tr><tr><td>GO:0000045</td><td>autophagosome assembly</td><td>biological_process</td><td>The formation of a double membrane-bounded structure, the autophagosome, that occurs when a specialized membrane sac, called the isolation membrane, starts to enclose a portion of the cytoplasm.</td><td>IBA</td></tr><tr><td>GO:0034727</td><td>piecemeal microautophagy of the nucleus</td><td>biological_process</td><td>Degradation of a cell nucleus by lysosomal microautophagy.</td><td>IBA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G25480</li><li>OGLUM01G26450</li><li>OBART01G22670</li><li>ORGLA01G0182100</li><li>KN539312.1_FG007</li><li>OMERI06G12060</li><li>OPUNC01G22860</li><li>OB01G32150</li><li>LPERR01G19460</li><li>TraesCS3A01G220600</li><li>BRADI_2g43480v3</li><li>TraesCS3B01G250900</li><li>HORVU3Hr1G054070</li><li>TRIUR3_32298</li><li>SETIT_000183mg</li><li>Zm00001d011347</li><li>SORBI_3003G219600</li><li>GSMUA_Achr2G06280_001</li><li>Dr19714</li><li>Solyc11g068930.1</li><li>GLYMA_12G010000</li><li>VIT_10s0003g01150</li><li>B456_009G163500</li><li>MTR_0003s0540</li><li>CCACVL1_09727</li><li>scaffold_800258.1</li><li>TCM_029712</li><li>PRUPE_4G060100</li><li>Csa_6G519540</li><li>DCAR_028009</li><li>PGSC0003DMG400019634</li><li>AT5G45900</li><li>TanjilG_16353</li><li>BVRB_8g181770</li><li>PHAVU_011G010700g</li><li>Tp57577_TGAC_v2_gene8345</li><li>Bra021999</li><li>MANES_17G054900</li><li>Bo2g127500</li><li>AMTR_s00059p00138250</li><li>PHYPA_028790</li><li>CHLRE_03g165215v5</li><li>OSTLU_49932</li><li>YHR171W</li><li>WBGene00010882</li><li>ENSG00000197548</li><li>FBgn0034366</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsATG7'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0614900'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g42850'}</td></tr><tr><th>Keyword</th><td>{'anther development', 'pollen', 'tapetum', 'anther', 'pollen development'}</td></tr><tr><th>Title</th><td>{'OsATG7 is required for autophagy-dependent lipid metabolism in rice postmeiotic anther development.'}</td></tr></table></td></tr><tr><th>funricegene_faminfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'ATG7'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0614900'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g42850'}</td></tr><tr><th>Name</th><td>{'ATG'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0614900</td></tr><tr><th>Description</th><td>NAD(P)-binding domain containing protein. (Os01t0614900-01);Similar to autophagy-related 7. (Os01t0614900-02)</td></tr><tr><th>Position</th><td>chr01:24380637..24386666</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>ATG7</td></tr><tr><th>CGSNL Gene Name</th><td>AUTOPHAGY ASSOCIATED GENE 7</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsATG7, Atg7</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>autophagy 7, OsAuTophaGy7, autophagy-related 7, AUTOPHAGY RELATED7</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsATG7'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0614900'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g42850'}</td></tr></table></td></tr></table>