<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> RD </td></tr><tr><th>Gene symbol synonym(s)</th><td> Rd DFR OsDFR OS-DFR </td></tr><tr><th>CGSNL Gene Name</th><td> RED PERICARP AND SEED COAT </td></tr><tr><th>Gene name synonym(s)</th><td> Red pericarp and seed coat dihydroflavonol 4-reductase dihydroflavonol-4-reductase </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Coloration - Anthocyanin Coloration - Others Seed Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0003824 - catalytic activity GO:0009416 - response to light stimulus GO:0009813 - flavonoid biosynthetic process GO:0010023 - proanthocyanidin biosynthetic process GO:0009812 - flavonoid metabolic process GO:0044237 - cellular metabolic process GO:0050662 - coenzyme binding GO:0051555 - flavonol biosynthetic process </td></tr><tr><th>Trait Ontology</th><td> TO:0000707 - pericarp color TO:0000487 - endosperm color TO:0000190 - seed coat color TO:0000486 - seed color TO:0000075 - light sensitivity </td></tr><tr><th>Plant Ontology</th><td> PO:0009088 - seed coat PO:0009089 - endosperm </td></tr><tr><th>RAP ID</th><td> Os01g0633500 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0633500</td></tr><tr><th>name</th><td>RED PERICARP AND SEED COAT</td></tr><tr><th>description</th><td>Similar to Dihydroflavonol reductase. (Os01t0633500-00)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>6112</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>25382714</td></tr><tr><th>end</th><td>25384678</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>RD</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0003939AAD</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>A0A0P0V5N7</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g44260.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS73318.1</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>Rd, DFR, OsDFR, OS-DFR</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR001509</td><td>Epimerase_deHydtase</td><td>NAD-dependent epimerase/dehydratase</td></tr><tr><td>IPR036291</td><td>NAD(P)-bd_dom_sf</td><td>NAD(P)-binding domain superfamily</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>16040</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0050662</td><td>coenzyme binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a coenzyme, any of various nonprotein organic cofactors that are required, in addition to an enzyme and a substrate, for an enzymatic reaction to proceed.</td><td>IEA</td></tr><tr><td>GO:0003824</td><td>catalytic activity</td><td>molecular_function</td><td>Catalysis of a biochemical reaction at physiological temperatures. In biologically catalyzed reactions, the reactants are known as substrates, and the catalysts are naturally occurring macromolecular substances known as enzymes. Enzymes possess specific binding sites for substrates, and are usually composed wholly or largely of protein, but RNA that has catalytic activity (ribozyme) is often also regarded as enzymatic.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORGLA01G0193000</li><li>BGIOSGA004056</li><li>OBART01G23740</li><li>ORUFI01G26600</li><li>OGLUM01G27570</li><li>OMERI01G21910</li><li>KN538828.1_FG034</li><li>OPUNC01G23860</li><li>OB01G33170</li><li>LPERR01G20410</li><li>TraesCS3B01G257900</li><li>TraesCS3A01G226600</li><li>TRIUR3_03992</li><li>BRADI_2g44480v3</li><li>F775_27635</li><li>HORVU3Hr1G056560</li><li>TraesCS3D01G224600</li><li>SETIT_005186mg</li><li>Dr05678</li><li>A4A49_37742</li><li>Bra027457</li><li>AT5G42800</li><li>BVRB_1g007170</li><li>DCAR_021485</li><li>MANES_18G022300</li><li>Solyc02g085020.2</li><li>PRUPE_1G376400</li><li>TanjilG_08193</li><li>PGSC0003DMG400003605</li><li>HannXRQ_Chr13g0407201</li></ul></td></tr><tr><th>syntenic_ortholog_one2many</th><td><ul><li>VIT_18s0001g12800</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os10g0477900</li><li>Os04g0630900</li><li>Os04g0630800</li><li>Os04g0630600</li><li>Os04g0630100</li><li>Os04g0630400</li><li>Os04g0630300</li><li>Os04g0631000</li><li>Os06g0683100</li><li>Os07g0602050</li><li>Os07g0598000</li><li>Os06g0651000</li><li>Os07g0601900</li><li>Os06g0651100</li><li>Os07g0601000</li><li>Os07g0601100</li><li>Os07g0602000</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsDfr|OsA1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0633500'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g44260'}</td></tr><tr><th>Keyword</th><td>{'seedling', 'transcription factor', 'salt'}</td></tr><tr><th>Title</th><td>{'Rice flavonoid pathway genes, OsDfr and OsAns, are induced by dehydration, high salt and ABA, and contain stress responsive promoter elements that interact with the transcription activator, OsC1-MYB'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0633500</td></tr><tr><th>Description</th><td>Dihydroflavonol-4-reductase (DFR), Proanthocyanidin synthesis (Os01t0633500-01)</td></tr><tr><th>Position</th><td>chr01:25382714..25384678</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>Rd, DFR</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>red pericarp and seed coat, dihydroflavonol 4-reductase</td></tr><tr><th>CGSNL Gene Symbol</th><td>RD</td></tr><tr><th>CGSNL Gene Name</th><td>RED PERICARP AND SEED COAT</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>Rd, DFR, OsDFR, OS-DFR</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>Red pericarp and seed coat, dihydroflavonol 4-reductase, dihydroflavonol-4-reductase</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsDfr|OsA1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0633500'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g44260'}</td></tr></table></td></tr></table>