<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> AP endonuclease </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character </td></tr><tr><th>Gene Ontology</th><td> GO:0003906 - DNA-(apurinic or apyrimidinic site) lyase activity GO:0005634 - nucleus GO:0046872 - metal ion binding GO:0008311 - double-stranded DNA specific 3'-5' exodeoxyribonuclease activity GO:0003677 - DNA binding GO:0006284 - base-excision repair </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0801000 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0801000</td></tr><tr><th>name</th><td>Os01g0801000</td></tr><tr><th>description</th><td>Similar to Apurinic endonuclease-redox protein (DNA-(apurinic or apyrimidinic site) lyase) (EC 4.2.99.18). (Os01t0801000-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>8472</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>33919497</td></tr><tr><th>end</th><td>33924589</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>UniParc</td><td><ul><li>UPI0000457D93</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q5VQJ9</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g58680.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015625662.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS74797.1</li><li>EEE55540.1</li><li>BAF06451.1</li><li>BAD68137.1</li><li>BAD68286.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK101426</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015770176.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4327960</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR003034</td><td>SAP_dom</td><td>SAP domain</td></tr><tr><td>IPR004808</td><td>AP_endonuc_1</td><td>AP endonuclease 1</td></tr><tr><td>IPR005135</td><td>Endo/exonuclease/phosphatase</td><td>Endonuclease/exonuclease/phosphatase</td></tr><tr><td>IPR020847</td><td>AP_endonuclease_F1_BS</td><td>AP endonuclease 1, binding site</td></tr><tr><td>IPR020848</td><td>AP_endonuclease_F1_CS</td><td>AP endonuclease 1, conserved site</td></tr><tr><td>IPR036361</td><td>SAP_dom_sf</td><td>SAP domain superfamily</td></tr><tr><td>IPR036691</td><td>Endo/exonu/phosph_ase_sf</td><td>Endonuclease/exonuclease/phosphatase superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0140078</td><td>class I DNA-(apurinic or apyrimidinic site) endonuclease activity</td><td>molecular_function</td><td>Catalysis of the cleavage of an AP site 3' of the baseless site by a beta-lyase mechanism, leaving an unsaturated aldehyde, termed a 3'-(4-hydroxy-5-phospho-2-pentenal) residue, and a 5'-phosphate.</td><td>IEA</td></tr><tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>Any molecular function by which a gene product interacts selectively and non-covalently with DNA (deoxyribonucleic acid).</td><td>IEA</td></tr><tr><td>GO:0046872</td><td>metal ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any metal ion.</td><td>IEA</td></tr><tr><td>GO:0008311</td><td>double-stranded DNA 3'-5' exodeoxyribonuclease activity</td><td>molecular_function</td><td>Catalysis of the sequential cleavage of mononucleotides from a free 3' terminus of a double-stranded DNA molecule.</td><td>IBA</td></tr><tr><td>GO:0006284</td><td>base-excision repair</td><td>biological_process</td><td>In base excision repair, an altered base is removed by a DNA glycosylase enzyme, followed by excision of the resulting sugar phosphate. The small gap left in the DNA helix is filled in by the sequential action of DNA polymerase and DNA ligase.</td><td>IBA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IBA</td></tr><tr><td>GO:0004519</td><td>endonuclease activity</td><td>molecular_function</td><td>Catalysis of the hydrolysis of ester linkages within nucleic acids by creating internal breaks.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>BGIOSGA004626</li><li>ORGLA01G0280100</li><li>KN540806.1_FG006</li><li>OB01G42910</li><li>OBART01G34100</li><li>OPUNC01G33130</li><li>OMERI01G30910</li><li>OGLUM01G38370</li><li>LPERR01G29460</li><li>ORUFI01G37410</li><li>TraesCS3B01G360300</li><li>TraesCS3A01G330200</li><li>F775_09511</li><li>TRIUR3_16333</li><li>BRADI_2g52540v3</li><li>HORVU3Hr1G078470</li><li>TraesCS3D01G323700</li><li>SORBI_3003G326500</li><li>SETIT_001057mg</li><li>Solyc04g016530.2</li><li>A4A49_12295</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0801100</li><li>Os12g0279600</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>POPTR_006G045600v3</li><li>POPTR_016G042000v3</li><li>GSBRNA2T00084635001</li><li>fgenesh2_kg.4__2307__AT2G41460.1</li><li>GSBRNA2T00123215001</li><li>Bo4g190930</li><li>Bo7g097720</li><li>fgenesh2_kg.4__2283__AT2G41460.1</li><li>GSBRNA2T00058401001</li><li>Bo3g035650</li><li>SELMODRAFT_103806</li><li>SELMODRAFT_99746</li><li>OSTLU_5639</li><li>OSTLU_5808</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0801000</td></tr><tr><th>Description</th><td>Similar to Apurinic endonuclease-redox protein (DNA-(apurinic or apyrimidinic site) lyase) (EC 4.2.99.18). (Os01t0801000-01)</td></tr><tr><th>Position</th><td>chr01:33919497..33924589</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td></td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>AP endonuclease</td></tr></table></td></tr></table>