<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsMYB3R-2 OsMYB3R2 MYB3R2 </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> O. sativa R1R2R3 MYB-2 R1R2R3 MYB-2 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0003682 - chromatin binding GO:0009414 - response to water deprivation GO:0009651 - response to salt stress GO:0009409 - response to cold GO:0003677 - DNA binding GO:0009751 - response to salicylic acid stimulus GO:0009609 - response to symbiotic bacterium </td></tr><tr><th>Trait Ontology</th><td> TO:0006001 - salt tolerance TO:0000276 - drought tolerance TO:0000303 - cold tolerance </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0841500 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0841500</td></tr><tr><th>name</th><td>O. sativa R1R2R3 MYB-2, R1R2R3 MYB-2</td></tr><tr><th>description</th><td>R1R2R3 MYB transcription factor, Stress tolerance (Os01t0841500-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9034</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>36122786</td></tr><tr><th>end</th><td>36128441</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>RAP-DB</td><td><ul><li>OsMYB3R-2, OsMYB3R2, MYB3R2</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0000DD8DE1</li></ul></td></tr><tr><td>Uniprot/Varsplic</td><td><ul><li>Q0JHU7-1</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q0JHU7</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g62410.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015616493.1</li><li>XP_015616500.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAD81765.1</li><li>BAF06681.1</li><li>EEE55651.1</li><li>BAS75160.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>FJ940212</li><li>AJ430052</li><li>FJ940219</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015761007.1</li><li>XM_015761014.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4327399</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsMYB3R-2, OsMYB3R2, MYB3R2</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR009057</td><td>Homeobox-like_sf</td><td>Homeobox-like domain superfamily</td></tr><tr><td>IPR017930</td><td>Myb_dom</td><td>Myb domain</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>1005</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:1902584</td><td>positive regulation of response to water deprivation</td><td>biological_process</td><td>Any process that activates or increases the frequency, rate or extent of response to water deprivation.</td><td>IMP</td></tr><tr><td>GO:0045893</td><td>positive regulation of transcription, DNA-templated</td><td>biological_process</td><td>Any process that activates or increases the frequency, rate or extent of cellular DNA-templated transcription.</td><td>IDA</td></tr><tr><td>GO:0006351</td><td>transcription, DNA-templated</td><td>biological_process</td><td>The cellular synthesis of RNA on a template of DNA.</td><td>IEA</td></tr><tr><td>GO:1901002</td><td>positive regulation of response to salt stress</td><td>biological_process</td><td>Any process that activates or increases the frequency, rate or extent of response to salt stress.</td><td>IMP</td></tr><tr><td>GO:0030154</td><td>cell differentiation</td><td>biological_process</td><td>The process in which relatively unspecialized cells, e.g. embryonic or regenerative cells, acquire specialized structural and/or functional features that characterize the cells, tissues, or organs of the mature organism or some other relatively stable phase of the organism's life history. Differentiation includes the processes involved in commitment of a cell to a specific fate and its subsequent development to the mature state.</td><td>IBA</td></tr><tr><td>GO:0044212</td><td>transcription regulatory region DNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a DNA region that regulates the transcription of a region of DNA, which may be a gene, cistron, or operon. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.</td><td>IBA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IDA</td></tr><tr><td>GO:0001135</td><td>RNA polymerase II transcription factor recruiting activity</td><td>molecular_function</td><td>The function of binding to an RNA polymerase II (RNAP II) transcription factor and recruiting it to the transcription machinery complex in order to modulate transcription by RNAP II.</td><td>IBA</td></tr><tr><td>GO:0000981</td><td>RNA polymerase II transcription factor activity, sequence-specific DNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a specific DNA sequence in order to modulate transcription by RNA polymerase II.</td><td>IBA</td></tr><tr><td>GO:0009409</td><td>response to cold</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a cold stimulus, a temperature stimulus below the optimal temperature for that organism.</td><td>IMP</td></tr><tr><td>GO:0043565</td><td>sequence-specific DNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with DNA of a specific nucleotide composition, e.g. GC-rich DNA binding, or with a specific sequence motif or type of DNA e.g. promotor binding or rDNA binding.</td><td>IDA</td></tr><tr><td>GO:0005739</td><td>mitochondrion</td><td>cellular_component</td><td>A semiautonomous, self replicating organelle that occurs in varying numbers, shapes, and sizes in the cytoplasm of virtually all eukaryotic cells. It is notably the site of tissue respiration.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORGLA01G0306500</li><li>OBART01G36770</li><li>BGIOSGA004774</li><li>OMERI01G33900</li><li>ORUFI01G40130</li><li>KN541294.1_FG003</li><li>OPUNC01G35700</li><li>OB01G45570</li><li>LPERR01G31650</li><li>BRADI_2g54640v3</li><li>TraesCS3A01G359200</li><li>HORVU3Hr1G086270</li><li>TRIUR3_33469</li><li>F775_12121</li><li>TraesCS3D01G353500</li><li>TraesCS3B01G391900</li><li>SORBI_3003G352200</li><li>SETIT_000842mg</li></ul></td></tr><tr><th>syntenic_ortholog_one2many</th><td><ul><li>Zm00001d012404</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os05g0459000</li><li>Os12g0238000</li><li>Os02g0271900</li><li>Os06g0162700</li><li>Os01g0855400</li><li>Os07g0627300</li><li>Os01g0229000</li><li>Os03g0236300</li><li>Os07g0627400</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>GSMUA_Achr6G05030_001</li><li>GSMUA_Achr7G10520_001</li><li>GSMUA_Achr10G26610_001</li><li>GLYMA_18G1811001</li><li>LR48_Vigan11g121000</li><li>PHAVU_010G012500g</li><li>B456_006G172800</li><li>Bo1g144510</li><li>GSBRNA2T00069535001</li><li>Bra001311</li><li>AT5G02320</li><li>GSBRNA2T00125127001</li><li>fgenesh1_pm.C_scaffold_6000124</li><li>PHAVU_008G102000g</li><li>GLYMA_03G082400</li><li>B456_001G249400</li><li>fgenesh1_pg.C_scaffold_3000839</li><li>LR48_Vigan03g138600</li><li>GSBRNA2T00080677001</li><li>Bo5g139340</li><li>GSBRNA2T00013972001</li><li>GSBRNA2T00111710001</li><li>Bra029762</li><li>Bo3g062020</li><li>GSBRNA2T00032377001</li><li>Bra034067</li><li>GLYMA_07G132200</li><li>AT3G09370</li><li>Bra009597</li><li>MTR_7g061330</li><li>Bra001312</li><li>GSBRNA2T00073268001</li><li>MTR_7g461410</li><li>Bo9g184660</li><li>ENSG00000101057</li><li>ENSG00000118513</li><li>ENSG00000185697</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsMYB3R-2'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0841500'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g62410'}</td></tr><tr><th>Keyword</th><td>{'chilling', 'transcription factor', 'mitosis', 'salt stress', 'cell cycle', 'cold tolerance', 'drought', 'salt'}</td></tr><tr><th>Title</th><td>{'Overexpression of an R1R2R3 MYB Gene, OsMYB3R-2, Increases Tolerance to Freezing, Drought, and Salt Stress in Transgenic Arabidopsis', 'Enhanced tolerance to chilling stress in OsMYB3R-2 transgenic rice is mediated by alteration in cell cycle and ectopic expression of stress genes'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0841500</td></tr><tr><th>Description</th><td>R1R2R3 MYB transcription factor, Stress tolerance (Os01t0841500-01)</td></tr><tr><th>Position</th><td>chr01:36122786..36128441</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>OsMYB3R-2, OsMYB3R2, MYB3R2</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>O. sativa R1R2R3 MYB-2, R1R2R3 MYB-2</td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsMYB3R-2, OsMYB3R2, MYB3R2</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>O. sativa R1R2R3 MYB-2, R1R2R3 MYB-2</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsMYB3R-2'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0841500'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g62410'}</td></tr></table></td></tr></table>