<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> HK3 </td></tr><tr><th>Gene symbol synonym(s)</th><td> OHK2 HK OsHK3 Ohk Crl2 Ohk2 OsHK3b </td></tr><tr><th>CGSNL Gene Name</th><td> HISTIDINE KINASE 3 </td></tr><tr><th>Gene name synonym(s)</th><td> histidine kinase 3 His kinase 3 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0000156 - two-component response regulator activity GO:0009737 - response to abscisic acid stimulus GO:0004673 - protein histidine kinase activity GO:0000155 - two-component sensor activity GO:0006979 - response to oxidative stress </td></tr><tr><th>Trait Ontology</th><td> TO:0002657 - oxidative stress TO:0000615 - abscisic acid sensitivity </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0923700 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0923700</td></tr><tr><th>name</th><td>HISTIDINE KINASE 3</td></tr><tr><th>description</th><td>Similar to Histidine kinase. (Os01t0923700-01);Similar to Histidine kinase. (Os01t0923700-02);Similar to Histidine kinase. (Os01t0923700-03);Histidine kinase, Abscisic acid (ABA)-induced antioxidant defense (Os01t0923700-04)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>10186</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>40403818</td></tr><tr><th>end</th><td>40409611</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>HK3</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000AA4A3</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>A1A696</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS75961.1</li><li>BAS75962.1</li><li>BAD88327.1</li><li>BAD88305.1</li><li>BAH91444.1</li><li>FAA00248.1</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g69920.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015649544.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AP014957</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015794058.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>9271913</li></ul></td></tr><tr><td>RAP-DB</td><td><ul><li>HK3, OHK2, HK, OsHK3, Ohk, Crl2, Ohk2, OsHK3b</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OHK2, HK, OsHK3, Ohk, Crl2, Ohk2, OsHK3b</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR001789</td><td>Sig_transdc_resp-reg_receiver</td><td>Signal transduction response regulator, receiver domain</td></tr><tr><td>IPR003594</td><td>HATPase_C</td><td>Histidine kinase/HSP90-like ATPase</td></tr><tr><td>IPR003661</td><td>HisK_dim/P</td><td>Signal transduction histidine kinase, dimerisation/phosphoacceptor domain</td></tr><tr><td>IPR004358</td><td>Sig_transdc_His_kin-like_C</td><td>Signal transduction histidine kinase-related protein, C-terminal</td></tr><tr><td>IPR005467</td><td>His_kinase_dom</td><td>Histidine kinase domain</td></tr><tr><td>IPR006189</td><td>CHASE_dom</td><td>CHASE domain</td></tr><tr><td>IPR011006</td><td>CheY-like_superfamily</td><td>CheY-like superfamily</td></tr><tr><td>IPR036097</td><td>HisK_dim/P_sf</td><td>Signal transduction histidine kinase, dimerisation/phosphoacceptor domain superfamily</td></tr><tr><td>IPR036890</td><td>HATPase_C_sf</td><td>Histidine kinase/HSP90-like ATPase superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0009414</td><td>response to water deprivation</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a water deprivation stimulus, prolonged deprivation of water.</td><td>IEA</td></tr><tr><td>GO:0010029</td><td>regulation of seed germination</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of seed germination.</td><td>IEA</td></tr><tr><td>GO:0010150</td><td>leaf senescence</td><td>biological_process</td><td>The process that occurs in a leaf near the end of its active life that is associated with the dismantling of cell components and membranes, loss of functional chloroplasts, and an overall decline in metabolism.</td><td>IEA</td></tr><tr><td>GO:0009736</td><td>cytokinin-activated signaling pathway</td><td>biological_process</td><td>A series of molecular signals generated by the binding of a cytokinin to a receptor, and ending with regulation of a downstream cellular process, e.g. transcription.</td><td>IEA</td></tr><tr><td>GO:0005886</td><td>plasma membrane</td><td>cellular_component</td><td>The membrane surrounding a cell that separates the cell from its external environment. It consists of a phospholipid bilayer and associated proteins.</td><td>IEA</td></tr><tr><td>GO:0016021</td><td>integral component of membrane</td><td>cellular_component</td><td>The component of a membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane.</td><td>IEA</td></tr><tr><td>GO:0080117</td><td>secondary growth</td><td>biological_process</td><td>Lateral growth of a plant axis (shoot axis or root) that is an increase in thickness resulting from formation of secondary vascular tissues by the vascular cambium.</td><td>IEA</td></tr><tr><td>GO:0000155</td><td>phosphorelay sensor kinase activity</td><td>molecular_function</td><td>Catalysis of the phosphorylation of a histidine residue in response to detection of an extracellular signal such as a chemical ligand or change in environment, to initiate a change in cell state or activity. The two-component sensor is a histidine kinase that autophosphorylates a histidine residue in its active site. The phosphate is then transferred to an aspartate residue in a downstream response regulator, to trigger a response.</td><td>IEA</td></tr><tr><td>GO:0034757</td><td>negative regulation of iron ion transport</td><td>biological_process</td><td>Any process that stops, prevents, or reduces the frequency, rate or extent of the directed movement of iron ions into, out of or within a cell, or between cells, by means of some agent such as a transporter or pore.</td><td>IEA</td></tr><tr><td>GO:0010087</td><td>phloem or xylem histogenesis</td><td>biological_process</td><td>The process whose specific outcome is the progression of phloem and/or xylem over time, from formation to the mature structure. An example of this process is found in Arabidopsis thaliana.</td><td>IEA</td></tr><tr><td>GO:0071329</td><td>cellular response to sucrose stimulus</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a sucrose stimulus.</td><td>IEA</td></tr><tr><td>GO:0010271</td><td>regulation of chlorophyll catabolic process</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the breakdown of chlorophyll.</td><td>IEA</td></tr><tr><td>GO:0071215</td><td>cellular response to abscisic acid stimulus</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an abscisic acid stimulus.</td><td>IEA</td></tr><tr><td>GO:0005622</td><td>intracellular</td><td>cellular_component</td><td>The living contents of a cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm.</td><td>IEA</td></tr><tr><td>GO:0042742</td><td>defense response to bacterium</td><td>biological_process</td><td>Reactions triggered in response to the presence of a bacterium that act to protect the cell or organism.</td><td>IEA</td></tr><tr><td>GO:0009909</td><td>regulation of flower development</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of flower development.</td><td>IEA</td></tr><tr><td>GO:0016036</td><td>cellular response to phosphate starvation</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of deprivation of phosphate.</td><td>IEA</td></tr><tr><td>GO:0048509</td><td>regulation of meristem development</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of meristem development, the biological process whose specific outcome is the progression of the meristem over time, from its formation to the mature structure.</td><td>IEA</td></tr><tr><td>GO:0070417</td><td>cellular response to cold</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a cold stimulus, a temperature stimulus below the optimal temperature for that organism.</td><td>IEA</td></tr><tr><td>GO:0009651</td><td>response to salt stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating an increase or decrease in the concentration of salt (particularly but not exclusively sodium and chloride ions) in the environment.</td><td>IEA</td></tr><tr><td>GO:0043424</td><td>protein histidine kinase binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with protein histidine kinase.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G45650</li><li>BGIOSGA000228</li><li>OBART01G42230</li><li>ORGLA01G0362400</li><li>OMERI01G39260</li><li>KN538700.1_FG004</li><li>OPUNC01G30720</li><li>OB01G51150</li><li>LPERR01G36790</li><li>SETIT_000168mg</li><li>SORBI_3003G407300</li><li>TraesCS3A01G433200</li><li>TRIUR3_22498</li><li>HORVU3Hr1G094870</li><li>TraesCS3D01G426600</li><li>BRADI_2g59127v3</li><li>TraesCS3B01G469000</li><li>F775_07796</li><li>Dr11278</li><li>Solyc05g015610.2</li><li>Csa_6G095330</li><li>Bo7g049980</li><li>DCAR_024678</li><li>PHAVU_002G173000g</li><li>PGSC0003DMG400034307</li><li>BVRB_6g150600</li><li>Tp57577_TGAC_v2_gene17770</li><li>fgenesh2_kg.1__2882__AT1G27320.1</li><li>CCACVL1_24082</li><li>LR48_Vigan01g106400</li><li>TCM_008737</li><li>PRUPE_1G224300</li><li>Bra030037</li><li>AT1G27320</li><li>AMTR_s00013p00077930</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os10g0362300</li><li>Os02g0738400</li><li>Os03g0717700</li><li>Os03g0701700</li><li>Os04g0169100</li><li>Os07g0259100</li><li>Os05g0155200</li><li>Os06g0654300</li><li>Os02g0820900</li><li>Os06g0183200</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>PHYPA_020620</li><li>SELMODRAFT_430773</li><li>SELMODRAFT_231095</li><li>SELMODRAFT_117801</li><li>SELMODRAFT_73679</li><li>PHYPA_008617</li><li>SELMODRAFT_158365</li><li>PHYPA_029750</li><li>SELMODRAFT_450507</li><li>CHLRE_13g571200v5</li><li>CHLRE_02g079750v5</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OHK2|OsHk3'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0923700'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g69920'}</td></tr><tr><th>Keyword</th><td>{'cytokinin'}</td></tr><tr><th>Title</th><td>{'Functional identification of OsHk6 as a homotypic cytokinin receptor in rice with preferential affinity for iP'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0923700</td></tr><tr><th>Description</th><td>Similar to Histidine kinase. (Os01t0923700-01);Similar to Histidine kinase. (Os01t0923700-02);Similar to Histidine kinase. (Os01t0923700-03);Histidine kinase, Abscisic acid (ABA)-induced antioxidant defense (Os01t0923700-04)</td></tr><tr><th>Position</th><td>chr01:40403818..40409611</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>HK3, OHK2, HK, OsHK3, Ohk, Crl2, Ohk2, OsHK3b</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>HISTIDINE KINASE 3, histidine kinase 3, His kinase 3</td></tr><tr><th>CGSNL Gene Symbol</th><td>HK3</td></tr><tr><th>CGSNL Gene Name</th><td>HISTIDINE KINASE 3</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OHK2, HK, OsHK3, Ohk, Crl2, Ohk2, OsHK3b</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>histidine kinase 3, His kinase 3</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OHK2|OsHk3'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0923700'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g69920'}</td></tr></table></td></tr></table>