<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsPDF1B </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> peptide deformylase 1B </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character </td></tr><tr><th>Gene Ontology</th><td> GO:0005506 - iron ion binding GO:0009570 - chloroplast stroma GO:0006412 - translation GO:0005739 - mitochondrion GO:0042586 - peptide deformylase activity </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0637600 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0637600</td></tr><tr><th>name</th><td>peptide deformylase 1B</td></tr><tr><th>description</th><td>Similar to Peptide deformylase, chloroplast precursor (EC 3.5.1.88) (PDF) (Polypeptide deformylase). (Os01t0637600-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>6154</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>25585914</td></tr><tr><th>end</th><td>25589162</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>Oryzabase</td><td><ul><li>OsPDF1B</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0000457EEF</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q5VNN5</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g45070.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015612309.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>EEE55058.1</li><li>BAD68576.1</li><li>BAD68940.1</li><li>BAF05583.1</li><li>BAS73349.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>EU213046</li><li>AK106980</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015756823.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326576</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR023635</td><td>Peptide_deformylase</td><td>Peptide deformylase</td></tr><tr><td>IPR036821</td><td>Peptide_deformylase_sf</td><td>Peptide deformylase superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0031365</td><td>N-terminal protein amino acid modification</td><td>biological_process</td><td>The alteration of the N-terminal amino acid residue in a protein.</td><td>IBA</td></tr><tr><td>GO:0006412</td><td>translation</td><td>biological_process</td><td>The cellular metabolic process in which a protein is formed, using the sequence of a mature mRNA or circRNA molecule to specify the sequence of amino acids in a polypeptide chain. Translation is mediated by the ribosome, and begins with the formation of a ternary complex between aminoacylated initiator methionine tRNA, GTP, and initiation factor 2, which subsequently associates with the small subunit of the ribosome and an mRNA or circRNA. Translation ends with the release of a polypeptide chain from the ribosome.</td><td>IEA</td></tr><tr><td>GO:0005739</td><td>mitochondrion</td><td>cellular_component</td><td>A semiautonomous, self replicating organelle that occurs in varying numbers, shapes, and sizes in the cytoplasm of virtually all eukaryotic cells. It is notably the site of tissue respiration.</td><td>IBA</td></tr><tr><td>GO:0043686</td><td>co-translational protein modification</td><td>biological_process</td><td>The process of covalently altering one or more amino acids in a protein after translation has begun but before the protein has been released from the ribosome.</td><td>IBA</td></tr><tr><td>GO:0046872</td><td>metal ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any metal ion.</td><td>IEA</td></tr><tr><td>GO:0009570</td><td>chloroplast stroma</td><td>cellular_component</td><td>The space enclosed by the double membrane of a chloroplast but excluding the thylakoid space. It contains DNA, ribosomes and some temporary products of photosynthesis.</td><td>IEA</td></tr><tr><td>GO:0042586</td><td>peptide deformylase activity</td><td>molecular_function</td><td>Catalysis of the reaction: formyl-L-methionyl peptide + H2O = formate + methionyl peptide.</td><td>IBA</td></tr><tr><td>GO:0018206</td><td>peptidyl-methionine modification</td><td>biological_process</td><td>The modification of peptidyl-methionine.</td><td>IBA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G26890</li><li>BGIOSGA001147</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0636600</li><li>Os01g0555800</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>GLYMA_19G061900</li><li>CCACVL1_25028</li><li>GLYMA_19G115400</li><li>GSBRNA2T00031625001</li><li>GSBRNA2T00135577001</li><li>CCACVL1_24985</li><li>SELMODRAFT_96122</li><li>SELMODRAFT_407843</li><li>SELMODRAFT_108250</li><li>SELMODRAFT_85358</li><li>CHLRE_01g039750v5</li><li>CHLRE_01g039650v5</li></ul></td></tr><tr><th>syntenic_ortholog_one2many</th><td><ul><li>VIT_14s0066g01070</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsPDF1B'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0637600'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g45070'}</td></tr><tr><th>Keyword</th><td>{'chloroplast', 'mitochondria', 'root'}</td></tr><tr><th>Title</th><td>{'Rice peptide deformylase PDF1B is crucial for development of chloroplasts'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0637600</td></tr><tr><th>Description</th><td>Similar to Peptide deformylase, chloroplast precursor (EC 3.5.1.88) (PDF) (Polypeptide deformylase). (Os01t0637600-01)</td></tr><tr><th>Position</th><td>chr01:25585914..25589162</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsPDF1B</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>peptide deformylase 1B</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsPDF1B'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0637600'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g45070'}</td></tr></table></td></tr></table>