<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> RAA1 </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsRAA1 </td></tr><tr><th>CGSNL Gene Name</th><td> ROOT ARCHITECTURE ASSOCIATED 1 </td></tr><tr><th>Gene name synonym(s)</th><td> root architecture associated-1 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Character as QTL - Root activity </td></tr><tr><th>Gene Ontology</th><td> GO:0005525 - GTP binding GO:0005634 - nucleus GO:0005737 - cytoplasm GO:0009629 - response to gravity GO:0009733 - response to auxin stimulus GO:0009739 - response to gibberellin stimulus GO:0009911 - positive regulation of flower development GO:0045841 - negative regulation of mitotic metaphase/anaphase transition GO:0048364 - root development </td></tr><tr><th>Trait Ontology</th><td> TO:0000656 - root development trait </td></tr><tr><th>Plant Ontology</th><td> PO:0007520 - root development stage </td></tr><tr><th>RAP ID</th><td> Os01g0257300 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0257300</td></tr><tr><th>name</th><td>ROOT ARCHITECTURE ASSOCIATED 1</td></tr><tr><th>description</th><td>Similar to FPF1. (Os01t0257300-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>2389</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>8586157</td></tr><tr><th>end</th><td>8586941</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>RAA1</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000A234C</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q9LGE3</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g15340.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015631985.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAF04545.1</li><li>EAZ11300.1</li><li>BAS71400.1</li><li>BAB07982.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK070626</li><li>AY659938</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015776499.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326229</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsRAA1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0045841</td><td>negative regulation of mitotic metaphase/anaphase transition</td><td>biological_process</td><td>Any process that stops, prevents, or reduces the frequency, rate or extent of the cell cycle process in which a cell progresses from metaphase to anaphase during mitosis, triggered by the activation of the anaphase promoting complex by Cdc20/Sleepy homolog which results in the degradation of Securin.</td><td>IDA</td></tr><tr><td>GO:0005525</td><td>GTP binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with GTP, guanosine triphosphate.</td><td>IDA</td></tr><tr><td>GO:0005737</td><td>cytoplasm</td><td>cellular_component</td><td>All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.</td><td>IEA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IEA</td></tr><tr><td>GO:2000280</td><td>regulation of root development</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of root development.</td><td>IMP</td></tr><tr><td>GO:0009733</td><td>response to auxin</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an auxin stimulus.</td><td>IDA</td></tr><tr><td>GO:0072686</td><td>mitotic spindle</td><td>cellular_component</td><td>A spindle that forms as part of mitosis. Mitotic and meiotic spindles contain distinctive complements of proteins associated with microtubules.</td><td>IDA</td></tr><tr><td>GO:0005515</td><td>protein binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).</td><td>IPI</td></tr><tr><td>GO:0009909</td><td>regulation of flower development</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of flower development.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G10830</li><li>ORGLA01G0089900</li><li>OBART01G09910</li><li>KN538708.1_FG093</li><li>OGLUM01G11330</li><li>OMERI01G09950</li><li>OPUNC01G09760</li><li>OB01G20070</li><li>LPERR01G09420</li><li>F775_19611</li><li>TraesCS3D01G181800</li><li>HORVU3Hr1G035680</li><li>TRIUR3_28721</li><li>TraesCSU01G198800</li><li>TraesCS3A01G176100</li><li>SETIT_003435mg</li><li>SORBI_3003G117600</li></ul></td></tr><tr><th>syntenic_ortholog_one2many</th><td><ul><li>Zm00001d040298</li><li>Zm00001d008325</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0933500</li><li>Os03g0346200</li><li>Os07g0671000</li><li>Os02g0460200</li><li>Os04g0282400</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>GSMUA_Achr2G00880_001</li><li>GSMUA_Achr11G15750_001</li><li>GSMUA_Achr8G30220_001</li><li>DCAR_000662</li><li>DCAR_004582</li><li>fgenesh2_kg.7__1037__AT4G31380.1</li><li>Bo9g011550</li><li>AT5G24860</li><li>GSBRNA2T00104007001</li><li>Bra036525</li><li>GSBRNA2T00147579001</li><li>GSBRNA2T00133565001</li><li>Bra011267</li><li>GSBRNA2T00077706001</li><li>Bo01097s020</li><li>DCAR_000663</li><li>BVRB_2g034800</li><li>Bra010240</li><li>Bra009799</li><li>GSBRNA2T00146011001</li><li>GSBRNA2T00103995001</li><li>Bo3g173980</li><li>GSBRNA2T00146015001</li><li>AT4G31380</li><li>scaffold_602590.1</li><li>Bo3g006730</li><li>MANES_03G002600</li><li>GLYMA_14G133300</li><li>PHAVU_001G058400g</li><li>BVRB_2g034840</li><li>Csa_2G379080</li><li>DCAR_004580</li><li>MANES_16G136500</li><li>AT5G10625</li><li>Bo7g115090</li><li>POPTR_006G276100v3</li><li>DCAR_004583</li><li>Bra011265</li><li>GSBRNA2T00130437001</li><li>A4A49_03860</li><li>Csa_3G881700</li><li>B456_009G068900</li><li>GSBRNA2T00023561001</li><li>B456_010G123400</li><li>Bo9g171910</li><li>Bo7g096470</li><li>GSBRNA2T00140142001</li><li>PHAVU_009G100300g</li><li>POPTR_018G005200v3</li><li>A4A49_65194</li><li>Bra009017</li><li>GSBRNA2T00062081001</li><li>GSBRNA2T00021866001</li><li>GSBRNA2T00135871001</li><li>GLYMA_17G202100</li><li>Bra011264</li><li>GLYMA_04G074800</li><li>Bo1g011370</li><li>Bra024011</li><li>GSBRNA2T00039999001</li><li>GSBRNA2T00101248001</li><li>GSBRNA2T00009584001</li><li>PHYPA_026261</li><li>PHYPA_022855</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsRAA1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0257300'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g15340'}</td></tr><tr><th>Keyword</th><td>{'seedling', 'primary root', 'cell division', 'auxin', 'flower', 'root development', 'growth', 'adventitious root', 'architecture', 'root architecture', 'meristem', 'cell cycle', 'leaf', 'sterile', 'lateral root', 'root'}</td></tr><tr><th>Title</th><td>{'Rice ROOT ARCHITECTURE ASSOCIATED1 binds the proteasome subunit RPT4 and is degraded in a D-box and proteasome-dependent manner', 'Overexpression of OsRAA1 causes pleiotropic phenotypes in transgenic rice plants, including altered leaf, flower, and root development and root response to gravity'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0257300</td></tr><tr><th>Description</th><td>Similar to FPF1. (Os01t0257300-01)</td></tr><tr><th>Position</th><td>chr01:8586157..8586941</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>RAA1</td></tr><tr><th>CGSNL Gene Name</th><td>ROOT ARCHITECTURE ASSOCIATED 1</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsRAA1</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>root architecture associated-1</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsRAA1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0257300'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g15340'}</td></tr></table></td></tr></table>