<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> NYC1 </td></tr><tr><th>Gene symbol synonym(s)</th><td> nyc1 OsNYC1 </td></tr><tr><th>CGSNL Gene Name</th><td> NON-YELLOW COLORING 1 </td></tr><tr><th>Gene name synonym(s)</th><td> Chlorophyl b degrading enzyme Chlase Non-Yellow Coloring 1 non-yellow coloring1 Probable chlorophyll(ide) b reductase NYC1 chloroplastic Protein NON-YELLOW COLORING 1 short-chain dehydrogenase/reductase NYC1 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Vegetative organ - Leaf Coloration - Chlorophyll </td></tr><tr><th>Gene Ontology</th><td> GO:0005488 - binding GO:0009535 - chloroplast thylakoid membrane GO:0009536 - plastid GO:0010150 - leaf senescence GO:0010304 - PSII associated light-harvesting complex II catabolic process GO:0015996 - chlorophyll catabolic process GO:0016021 - integral to membrane GO:0016491 - oxidoreductase activity GO:0034256 - chlorophyll(ide) b reductase activity GO:0042170 - plastid membrane GO:0055114 - oxidation reduction </td></tr><tr><th>Trait Ontology</th><td> TO:0000599 - enzyme activity TO:0000495 - chlorophyll content TO:0000249 - leaf senescence TO:0002712 - stay green trait </td></tr><tr><th>Plant Ontology</th><td> PO:0001054 - 4 leaf senescence stage PO:0009025 - vascular leaf PO:0009037 - lemma PO:0009038 - palea PO:0020104 - leaf sheath PO:0020122 - inflorescence axis </td></tr><tr><th>RAP ID</th><td> Os01g0227100 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0227100</td></tr><tr><th>name</th><td>NON-YELLOW COLORING 1</td></tr><tr><th>description</th><td>Chlorophyll b reductase, Leaf senescence (Os01t0227100-01);Similar to oxidoreductase. (Os01t0227100-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>1948</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>7024297</td></tr><tr><th>end</th><td>7030591</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>NYC1</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI000049E7B7</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q5N800</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS71129.1</li><li>EEE54154.1</li><li>BAF04382.1</li><li>BAD82407.1</li><li>BAS71128.1</li><li>BAD81513.1</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g12710.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015621887.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK068523</li><li>AB255025</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015766401.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4327178</li></ul></td></tr><tr><td>RAP-DB</td><td><ul><li>NYC1, nyc1, OsNYC1</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>nyc1, OsNYC1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR002347</td><td>SDR_fam</td><td>Short-chain dehydrogenase/reductase SDR</td></tr><tr><td>IPR036291</td><td>NAD(P)-bd_dom_sf</td><td>NAD(P)-binding domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0009535</td><td>chloroplast thylakoid membrane</td><td>cellular_component</td><td>The pigmented membrane of a chloroplast thylakoid. An example of this component is found in Arabidopsis thaliana.</td><td>IEA</td></tr><tr><td>GO:0015996</td><td>chlorophyll catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of chlorophyll, any compound of magnesium complexed in a porphyrin (tetrapyrrole) ring and which functions as a photosynthetic pigment, into less complex products.</td><td>IMP</td></tr><tr><td>GO:0016021</td><td>integral component of membrane</td><td>cellular_component</td><td>The component of a membrane consisting of the gene products and protein complexes having at least some part of their peptide sequence embedded in the hydrophobic region of the membrane.</td><td>IEA</td></tr><tr><td>GO:0034256</td><td>chlorophyll(ide) b reductase activity</td><td>molecular_function</td><td>Catalysis of the reaction: 71-hydroxychlorophyll(ide) a + NAD(P)+ = chlorophyll(ide) b + NAD(P)H + H+.</td><td>IBA</td></tr><tr><td>GO:0010304</td><td>PSII associated light-harvesting complex II catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of one or more components of the light-harvesting complex of photosystem II.</td><td>IMP</td></tr><tr><td>GO:0042170</td><td>plastid membrane</td><td>cellular_component</td><td>Either of the lipid bilayers that surround a plastid and form the plastid envelope.</td><td>RCA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>BGIOSGA003046</li><li>ORUFI01G08790</li><li>ORGLA01G0071700</li><li>OBART01G07950</li><li>KN538689.1_FG053</li><li>OGLUM01G09120</li><li>OMERI01G07970</li><li>OPUNC01G07810</li><li>OB01G18200</li><li>LPERR01G07550</li><li>BRADI_2g07550v3</li><li>HORVU3Hr1G030890</li><li>F775_29909</li><li>TraesCS3B01G179100</li><li>TRIUR3_24422</li><li>TraesCS3D01G159800</li><li>TraesCS3A01G151900</li><li>Zm00001d039312</li><li>SETIT_001132mg</li><li>Dr09305</li><li>Csa_1G250130</li><li>AMTR_s00109p00036750</li><li>DCAR_012071</li><li>BVRB_8g196400</li><li>PHAVU_004G163900g</li><li>Tp57577_TGAC_v2_gene35643</li><li>PGSC0003DMG400030219</li><li>VIT_11s0016g03890</li><li>A4A49_06386</li><li>CCACVL1_07934</li><li>LR48_Vigan10g015000</li><li>Solyc07g024000.2</li><li>MTR_6g088500</li><li>POPTR_018G081200v3</li><li>PRUPE_7G232100</li><li>TCM_037874</li><li>B456_009G025000</li><li>AT4G13250</li><li>fgenesh2_kg.9__30__AT4G13250.1</li><li>PHYPA_012256</li><li>CHLRE_12g517700v5</li><li>OSTLU_39837</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os03g0654600</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>FBgn0026268</li><li>FBgn0031775</li><li>FBgn0030073</li><li>FBgn0263830</li><li>FBgn0038946</li><li>FBgn0036380</li><li>ENSCSAVG00000002588</li><li>FBgn0030332</li><li>FBgn0038878</li><li>ENSCSAVG00000010251</li><li>FBgn0069973</li><li>ENSCSAVG00000006001</li><li>ENSCSAVG00000010204</li><li>Gasu_45810</li><li>Gasu_03130</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'NYC1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0227100'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g12710'}</td></tr><tr><th>Keyword</th><td>{'chloroplast', 'leaf', 'senescence'}</td></tr><tr><th>Title</th><td>{'Rice NON-YELLOW COLORING1 is involved in light-harvesting complex II and grana degradation during leaf senescence', 'Two short-chain dehydrogenase/reductases, NON-YELLOW COLORING 1 and NYC1-LIKE, are required for chlorophyll b and light-harvesting complex II degradation during senescence in rice'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0227100</td></tr><tr><th>Description</th><td>Chlorophyll b reductase, Leaf senescence (Os01t0227100-01);Similar to oxidoreductase. (Os01t0227100-02)</td></tr><tr><th>Position</th><td>chr01:7024297..7030591</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>NYC1, nyc1, OsNYC1</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>NON-YELLOW COLORING 1, Chlorophyl b degrading enzyme, Chlase, Non-Yellow Coloring 1, non-yellow coloring1, Probable chlorophyll(ide) b reductase NYC1, chloroplastic, Protein NON-YELLOW COLORING 1, short-chain dehydrogenase/reductase NYC1</td></tr><tr><th>CGSNL Gene Symbol</th><td>NYC1</td></tr><tr><th>CGSNL Gene Name</th><td>NON-YELLOW COLORING 1</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>nyc1, OsNYC1</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>Chlorophyl b degrading enzyme, Chlase, Non-Yellow Coloring 1, non-yellow coloring1, Probable chlorophyll(ide) b reductase NYC1, chloroplastic, Protein NON-YELLOW COLORING 1, short-chain dehydrogenase/reductase NYC1</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'NYC1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0227100'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g12710'}</td></tr></table></td></tr></table>