<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> OSPOLA Ospolalpha </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> DNA polymerase alpha catalytic subunit catalytic subunit of DNA polymerase alpha </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character </td></tr><tr><th>Gene Ontology</th><td> GO:0006270 - DNA replication initiation GO:0001882 - nucleoside binding GO:0003677 - DNA binding GO:0006273 - lagging strand elongation GO:0046872 - metal ion binding GO:0003887 - DNA-directed DNA polymerase activity GO:0005634 - nucleus GO:0003896 - DNA primase activity GO:0051539 - 4 iron, 4 sulfur cluster binding GO:0000166 - nucleotide binding </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0868300 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0868300</td></tr><tr><th>name</th><td>DNA polymerase alpha catalytic subunit, catalytic subunit of DNA polymerase alpha</td></tr><tr><th>description</th><td>Similar to DNA polymerase alpha catalytic subunit (EC 2.7.7.7). (Os01t0868300-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9401</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>37625916</td></tr><tr><th>end</th><td>37636099</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>Oryzabase</td><td><ul><li>OSPOLA, Ospolalpha</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0000668816</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>O48653</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g64820.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015624973.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS75408.1</li><li>BAD81999.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AB004461</li></ul></td></tr><tr><td>KEGG_Enzyme</td><td><ul><li>00240+2.7.7.7</li><li>00230+2.7.7.7</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015769487.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4325052</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR006133</td><td>DNA-dir_DNA_pol_B_exonuc</td><td>DNA-directed DNA polymerase, family B, exonuclease domain</td></tr><tr><td>IPR006134</td><td>DNA-dir_DNA_pol_B_multi_dom</td><td>DNA-directed DNA polymerase, family B, multifunctional domain</td></tr><tr><td>IPR006172</td><td>DNA-dir_DNA_pol_B</td><td>DNA-directed DNA polymerase, family B</td></tr><tr><td>IPR012337</td><td>RNaseH-like_sf</td><td>Ribonuclease H-like superfamily</td></tr><tr><td>IPR015088</td><td>Znf_DNA-dir_DNA_pol_B_alpha</td><td>Zinc finger, DNA-directed DNA polymerase, family B, alpha</td></tr><tr><td>IPR017964</td><td>DNA-dir_DNA_pol_B_CS</td><td>DNA-directed DNA polymerase, family B, conserved site</td></tr><tr><td>IPR023211</td><td>DNA_pol_palm_dom_sf</td><td>DNA polymerase, palm domain superfamily</td></tr><tr><td>IPR024647</td><td>DNA_pol_a_cat_su_N</td><td>DNA polymerase alpha catalytic subunit, N-terminal domain</td></tr><tr><td>IPR036397</td><td>RNaseH_sf</td><td>Ribonuclease H superfamily</td></tr><tr><td>IPR038256</td><td>Pol_alpha_znc_sf</td><td>DNA polymerase alpha, zinc finger domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0000166</td><td>nucleotide binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a nucleotide, any compound consisting of a nucleoside that is esterified with (ortho)phosphate or an oligophosphate at any hydroxyl group on the ribose or deoxyribose.</td><td>IEA</td></tr><tr><td>GO:0006303</td><td>double-strand break repair via nonhomologous end joining</td><td>biological_process</td><td>The repair of a double-strand break in DNA in which the two broken ends are rejoined with little or no sequence complementarity. Information at the DNA ends may be lost due to the modification of broken DNA ends. This term covers instances of separate pathways, called classical (or canonical) and alternative nonhomologous end joining (C-NHEJ and A-NHEJ). These in turn may further branch into sub-pathways, but evidence is still unclear.</td><td>IBA</td></tr><tr><td>GO:0005658</td><td>alpha DNA polymerase:primase complex</td><td>cellular_component</td><td>A complex of four polypeptides, comprising large and small DNA polymerase alpha subunits and two primase subunits, which catalyzes the synthesis of an RNA primer on the lagging strand of replicating DNA; the smaller of the two primase subunits alone can catalyze oligoribonucleotide synthesis.</td><td>IBA</td></tr><tr><td>GO:0006273</td><td>lagging strand elongation</td><td>biological_process</td><td>The synthesis of DNA from a template strand in a net 3' to 5' direction. Lagging strand DNA elongation proceeds by discontinuous synthesis of short stretches of DNA, known as Okazaki fragments, from RNA primers; these fragments are then joined by DNA ligase. Although each segment of nascent DNA is synthesized in the 5' to 3' direction, the overall direction of lagging strand synthesis is 3' to 5', mirroring the progress of the replication fork.</td><td>IDA</td></tr><tr><td>GO:0051539</td><td>4 iron, 4 sulfur cluster binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a 4 iron, 4 sulfur (4Fe-4S) cluster; this cluster consists of four iron atoms, with the inorganic sulfur atoms found between the irons and acting as bridging ligands.</td><td>IEA</td></tr><tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>Any molecular function by which a gene product interacts selectively and non-covalently with DNA (deoxyribonucleic acid).</td><td>IEA</td></tr><tr><td>GO:0003896</td><td>DNA primase activity</td><td>molecular_function</td><td>Catalysis of the synthesis of a short RNA primer on a DNA template, providing a free 3'-OH that can be extended by DNA-directed DNA polymerases.</td><td>IDA</td></tr><tr><td>GO:0006272</td><td>leading strand elongation</td><td>biological_process</td><td>The synthesis of DNA from a template strand in the 5' to 3' direction; leading strand elongation is continuous as it proceeds in the same direction as the replication fork.</td><td>IBA</td></tr><tr><td>GO:0001882</td><td>nucleoside binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a nucleoside, a compound consisting of a purine or pyrimidine nitrogenous base linked either to ribose or deoxyribose.</td><td>IEA</td></tr><tr><td>GO:0006270</td><td>DNA replication initiation</td><td>biological_process</td><td>The process in which DNA-dependent DNA replication is started; this begins with the ATP dependent loading of an initiator complex onto the DNA, this is followed by DNA melting and helicase activity. In bacteria, the gene products that enable the helicase activity are loaded after the initial melting and in archaea and eukaryotes, the gene products that enable the helicase activity are inactive when they are loaded and subsequently activate.</td><td>IDA</td></tr><tr><td>GO:0003887</td><td>DNA-directed DNA polymerase activity</td><td>molecular_function</td><td>Catalysis of the reaction: deoxynucleoside triphosphate + DNA(n) = diphosphate + DNA(n+1); the synthesis of DNA from deoxyribonucleotide triphosphates in the presence of a DNA template and a 3'hydroxyl group.</td><td>IDA</td></tr><tr><td>GO:0046872</td><td>metal ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any metal ion.</td><td>IEA</td></tr><tr><td>GO:0009965</td><td>leaf morphogenesis</td><td>biological_process</td><td>The process in which the anatomical structures of the leaf are generated and organized.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G41980</li><li>ONIVA01G43430</li><li>BGIOSGA000426</li><li>OBART01G38610</li><li>ORGLA01G0324300</li><li>OMERI01G35490</li><li>KN538837.1_FG012</li><li>OPUNC01G37330</li><li>OB01G47300</li><li>LPERR01G33130</li><li>BRADI_2g56160v3</li><li>HORVU3Hr1G082930</li><li>F775_04971</li><li>SORBI_3003G369200</li><li>SETIT_000033mg</li><li>Zm00001d042713</li><li>AT5G67100</li><li>VIT_07s0031g00470</li><li>POPTR_007G045300v3</li><li>TanjilG_06189</li><li>Solyc02g093300.2</li><li>Csa_2G278160</li><li>DCAR_025744</li><li>HannXRQ_Chr14g0449061</li><li>MTR_5g017070</li><li>PRUPE_7G131600</li><li>CCACVL1_26393</li><li>BVRB_008590</li><li>PHAVU_002G017900g</li><li>B456_007G214600</li><li>MANES_13G122900</li><li>TCM_000647</li><li>GLYMA_01G187400</li><li>AMTR_s00017p00221560</li><li>PHYPA_000926</li><li>OSTLU_297</li><li>CHLRE_04g214350v5</li><li>FBgn0259113</li><li>ENSCSAVG00000000726</li><li>ENSG00000101868</li><li>YNL102W</li><li>WBGene00012936</li><li>Gasu_18650</li><li>CHC_T00010052001</li><li>CMI176C</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os11g0186400</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0868300</td></tr><tr><th>Description</th><td>Similar to DNA polymerase alpha catalytic subunit (EC 2.7.7.7). (Os01t0868300-01)</td></tr><tr><th>Position</th><td>chr01:37625916..37636099</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OSPOLA, Ospolalpha</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>DNA polymerase alpha catalytic subunit, catalytic subunit of DNA polymerase alpha</td></tr></table></td></tr></table>