<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> VP1 </td></tr><tr><th>Gene symbol synonym(s)</th><td> Vp1* (OSVP1) OsVP1 OsVp1 Vp1 ABI3 OsABI3 OsLFL4 OsVp-1 OSVP1 </td></tr><tr><th>CGSNL Gene Name</th><td> VIVIPAROUS 1 </td></tr><tr><th>Gene name synonym(s)</th><td> Viviparous-1 Protein viviparous homolog B3 domain-containing protein VP1 Transcription activator VP1-rice LEAFY COTYLEDON 2 and FUSCA 3-LIKE 4 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Seed - Physiological traits - Dormancy Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0009845 - seed germination GO:0009737 - response to abscisic acid stimulus GO:0017163 - basal transcription repressor activity GO:0003677 - DNA binding GO:0005634 - nucleus GO:0006350 - transcription GO:0006355 - regulation of transcription, DNA-dependent GO:0009738 - abscisic acid mediated signaling GO:0009790 - embryonic development GO:0045449 - regulation of transcription GO:0009414 - response to water deprivation </td></tr><tr><th>Trait Ontology</th><td> TO:0000619 - vivipary TO:0000615 - abscisic acid sensitivity TO:0000253 - seed dormancy TO:0000276 - drought tolerance </td></tr><tr><th>Plant Ontology</th><td> PO:0009010 - seed </td></tr><tr><th>RAP ID</th><td> Os01g0911700 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0911700</td></tr><tr><th>name</th><td>VIVIPAROUS 1</td></tr><tr><th>description</th><td>Similar to Regulatory protein viviparous-1. (Os01t0911700-01);Transcription activator VP1-rice. (Os01t0911700-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9993</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>39723171</td></tr><tr><th>end</th><td>39726984</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>VP1</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI000049EA4B</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>P37398</li></ul></td></tr><tr><td>Uniprot/Varsplic</td><td><ul><li>P37398-1</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g68370.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015629318.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAA04066.1</li><li>BAS75842.1</li><li>BAD82432.1</li><li>BAS75841.1</li><li>BAF07076.2</li><li>BAD82433.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK105441</li><li>AK073805</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015773832.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4324314</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>Vp1* (OSVP1), OsVP1, OsVp1, Vp1, ABI3, OsLFL4, OsVp-1, OSVP1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR003340</td><td>B3_DNA-bd</td><td>B3 DNA binding domain</td></tr><tr><td>IPR015300</td><td>DNA-bd_pseudobarrel_sf</td><td>DNA-binding pseudobarrel domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0009733</td><td>response to auxin</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an auxin stimulus.</td><td>IEA</td></tr><tr><td>GO:0001076</td><td>transcription factor activity, RNA polymerase II transcription factor binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with an RNA polymerase II transcription factor, which may be a single protein or a complex, in order to modulate transcription. A protein binding transcription factor may or may not also interact with the template nucleic acid (either DNA or RNA) as well.</td><td>IEA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IEP</td></tr><tr><td>GO:0009738</td><td>abscisic acid-activated signaling pathway</td><td>biological_process</td><td>A series of molecular signals generated by the binding of the plant hormone abscisic acid (ABA) to a receptor, and ending with modulation of a cellular process, e.g. transcription.</td><td>IEA</td></tr><tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>Any molecular function by which a gene product interacts selectively and non-covalently with DNA (deoxyribonucleic acid).</td><td>IEA</td></tr><tr><td>GO:0009793</td><td>embryo development ending in seed dormancy</td><td>biological_process</td><td>The process whose specific outcome is the progression of the embryo over time, from zygote formation to the end of seed dormancy. An example of this process is found in Arabidopsis thaliana.</td><td>IEA</td></tr><tr><td>GO:0009657</td><td>plastid organization</td><td>biological_process</td><td>A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of a plastid.</td><td>IEA</td></tr><tr><td>GO:0031930</td><td>mitochondria-nucleus signaling pathway</td><td>biological_process</td><td>A series of molecular signals that forms a pathway of communication from the mitochondria to the nucleus and initiates cellular changes in response to changes in mitochondrial function.</td><td>IEA</td></tr><tr><td>GO:0006351</td><td>transcription, DNA-templated</td><td>biological_process</td><td>The cellular synthesis of RNA on a template of DNA.</td><td>IEA</td></tr><tr><td>GO:0005829</td><td>cytosol</td><td>cellular_component</td><td>The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.</td><td>IEA</td></tr><tr><td>GO:0045893</td><td>positive regulation of transcription, DNA-templated</td><td>biological_process</td><td>Any process that activates or increases the frequency, rate or extent of cellular DNA-templated transcription.</td><td>IEA</td></tr><tr><td>GO:0003700</td><td>DNA-binding transcription factor activity</td><td>molecular_function</td><td>A protein or a member of a complex that interacts selectively and non-covalently with a specific DNA sequence (sometimes referred to as a motif) within the regulatory region of a gene in order to modulate transcription. Regulatory regions include promoters (proximal and distal) and enhancers. Genes are transcriptional units, and include bacterial operons.</td><td>IEA</td></tr><tr><td>GO:0005515</td><td>protein binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).</td><td>IPI</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G44750</li><li>BGIOSGA000276</li><li>ONIVA01G46510</li><li>ORGLA01G0353400</li><li>OBART01G41410</li><li>OMERI01G38380</li><li>KN538752.1_FG020</li><li>OB01G50120</li><li>LPERR01G35880</li><li>TraesCS3D01G412800</li><li>TRIUR3_22986</li><li>F775_43744</li><li>HORVU3Hr1G092690</li><li>TraesCS3A01G417300</li><li>BRADI_2g58390v3</li><li>TraesCS3B01G452200</li><li>Zm00001d042396</li><li>SETIT_000497mg</li><li>Dr13878</li><li>POPTR_002G252000v3</li><li>BVRB_7g167370</li><li>B456_001G184300</li><li>AT3G24650</li><li>Csa_7G336510</li><li>VIT_07s0005g05400</li><li>MTR_7g059330</li><li>CCACVL1_27470</li><li>PRUPE_8G207700</li><li>PHAVU_008G122100g</li><li>TCM_003056</li><li>LR48_Vigan317s001200</li><li>Bo3g083650</li><li>Tp57577_TGAC_v2_gene23267</li><li>fgenesh1_pg.C_scaffold_3002399</li><li>Bra013248</li><li>AMTR_s00049p00034730</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0713600</li><li>Os04g0676600</li><li>Os04g0676650</li><li>Os08g0101000</li><li>Os07g0679700</li><li>Os07g0563300</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OSVP1|VP1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0911700'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g68370'}</td></tr><tr><th>Keyword</th><td>{'seedling', 'flower', 'transcription factor', 'growth', 'ABA', 'seedlings', 'seed', 'dormancy', ' ABA '}</td></tr><tr><th>Title</th><td>{'Molecular cloning of Sdr4, a regulator involved in seed dormancy and domestication of rice', 'The Seed-Specific Transcription Factor VP1 (OSVP1) Is Expressed in Rice Suspension-Cultured Cells', 'Short, direct repeats (SDRs)-mediated post-transcriptional processing of a transcription factor gene OsVP1 in rice (Oryza sativa)', 'A bZIP factor, TRAB1, interacts with VP1 and mediates abscisic acid-induced transcription', 'Sequence and functional analyses of the rice gene homologous to the maize Vp1', 'Rice PcG gene OsEMF2b controls seed dormancy and seedling growth by regulating the expression of OsVP1.', 'Regulation of the Osem gene by abscisic acid and the transcriptional activator VP1: analysis of cis-acting promoter elements required for regulation by abscisic acid and VP1'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0911700</td></tr><tr><th>Description</th><td>Similar to Regulatory protein viviparous-1. (Os01t0911700-01);Transcription activator VP1-rice. (Os01t0911700-02)</td></tr><tr><th>Position</th><td>chr01:39723171..39726984</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>VP1</td></tr><tr><th>CGSNL Gene Name</th><td>VIVIPAROUS 1</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>Vp1* (OSVP1), OsVP1, OsVp1, Vp1, ABI3, OsLFL4, OsVp-1, OSVP1</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>Viviparous-1, Protein viviparous homolog, B3 domain-containing protein VP1, Transcription activator VP1-rice, LEAFY COTYLEDON 2 and FUSCA 3-LIKE 4</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OSVP1|VP1'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0911700'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g68370'}</td></tr></table></td></tr></table>