<table style="width:100%" class="table table-striped" border="1"><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0328700</td></tr><tr><th>name</th><td>Os01g0328700</td></tr><tr><th>description</th><td>FAD-dependent pyridine nucleotide-disulphide oxidoreductase domain containing protein. (Os01t0328700-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>3340</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>12655016</td></tr><tr><th>end</th><td>12658349</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>UniParc</td><td><ul><li>UPI00000A397B</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q9ASP4</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g22520.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015611017.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAB39219.1</li><li>BAS71896.1</li><li>BAF04832.1</li><li>EAZ11701.1</li></ul></td></tr><tr><td>KEGG_Enzyme</td><td><ul><li>00260+1.8.1.4</li><li>00640+1.8.1.4</li><li>00620+1.8.1.4</li><li>00020+1.8.1.4</li><li>00010+1.8.1.4</li><li>00280+1.8.1.4</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015755531.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326980</li></ul></td></tr><tr><td>notGramene_Plant_Reactome</td><td><ul><li>R-OSA-1119533.1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>pathways</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ancestors</th><td><ul><li>2744345</li><li>2961031</li><li>1119533</li><li>5368291</li><li>1119443</li><li>1119351</li><li>1119312</li></ul></td></tr><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>link</th><th>name</th></tr><tr><td>R-OSA-1120492</td><td>http://plantreactome.gramene.org/PathwayBrowser/#/R-OSA-1120492</td><td>2-oxoglutarate + coA-SH + NAD+ => succinyl-CoA + CO2 + NADH + H+</td></tr><tr><td>R-OSA-1169121</td><td>http://plantreactome.gramene.org/PathwayBrowser/#/R-OSA-1169121</td><td>pyruvate + CoA-SH + NAD+ => acetyl-CoA + CO2 + NADH + H+</td></tr><tr><td>R-OSA-8987464</td><td>http://plantreactome.gramene.org/PathwayBrowser/#/R-OSA-8987464</td><td>glycine+NAD+THF -> 5,10-ch2-THF+CO2+NH3+NADH</td></tr></table></td></tr></table></td></tr><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR004099</td><td>Pyr_nucl-diS_OxRdtase_dimer</td><td>Pyridine nucleotide-disulphide oxidoreductase, dimerisation domain</td></tr><tr><td>IPR006258</td><td>Lipoamide_DH</td><td>Dihydrolipoamide dehydrogenase</td></tr><tr><td>IPR012999</td><td>Pyr_OxRdtase_I_AS</td><td>Pyridine nucleotide-disulphide oxidoreductase, class I, active site</td></tr><tr><td>IPR016156</td><td>FAD/NAD-linked_Rdtase_dimer_sf</td><td>FAD/NAD-linked reductase, dimerisation domain superfamily</td></tr><tr><td>IPR023753</td><td>FAD/NAD-binding_dom</td><td>FAD/NAD(P)-binding domain</td></tr><tr><td>IPR036188</td><td>FAD/NAD-bd_sf</td><td>FAD/NAD(P)-binding domain superfamily</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>1100</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0004148</td><td>dihydrolipoyl dehydrogenase activity</td><td>molecular_function</td><td>Catalysis of the reaction: protein N6-(dihydrolipoyl)lysine + NAD+ = protein N6-(lipoyl)lysine + NADH + H+.</td><td>IEA</td></tr><tr><td>GO:0045454</td><td>cell redox homeostasis</td><td>biological_process</td><td>Any process that maintains the redox environment of a cell or compartment within a cell.</td><td>IEA</td></tr><tr><td>GO:0005623</td><td>cell</td><td>cellular_component</td><td>The basic structural and functional unit of all organisms. Includes the plasma membrane and any external encapsulating structures such as the cell wall and cell envelope.</td><td>IEA</td></tr><tr><td>GO:0050660</td><td>flavin adenine dinucleotide binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with FAD, flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes, in either the oxidized form, FAD, or the reduced form, FADH2.</td><td>IEA</td></tr><tr><td>GO:0009055</td><td>electron transfer activity</td><td>molecular_function</td><td>Any molecular entity that serves as an electron acceptor and electron donor in an electron transport chain. An electron transport chain is a process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient.</td><td>IEA</td></tr><tr><td>GO:0003676</td><td>nucleic acid binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any nucleic acid.</td><td>IEA</td></tr><tr><td>GO:0005524</td><td>ATP binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator.</td><td>IEA</td></tr><tr><td>GO:0008026</td><td>ATP-dependent helicase activity</td><td>molecular_function</td><td>Catalysis of the reaction: ATP + H2O = ADP + phosphate, to drive the unwinding of a DNA or RNA helix.</td><td>IEA</td></tr><tr><td>GO:0008375</td><td>acetylglucosaminyltransferase activity</td><td>molecular_function</td><td>Catalysis of the transfer of an N-acetylglucosaminyl residue from UDP-N-acetyl-glucosamine to a sugar.</td><td>IEA</td></tr><tr><td>GO:0008408</td><td>3'-5' exonuclease activity</td><td>molecular_function</td><td>Catalysis of the hydrolysis of ester linkages within nucleic acids by removing nucleotide residues from the 3' end.</td><td>IEA</td></tr><tr><td>GO:0016746</td><td>transferase activity, transferring acyl groups</td><td>molecular_function</td><td>Catalysis of the transfer of an acyl group from one compound (donor) to another (acceptor).</td><td>IEA</td></tr><tr><td>GO:0006260</td><td>DNA replication</td><td>biological_process</td><td>The cellular metabolic process in which a cell duplicates one or more molecules of DNA. DNA replication begins when specific sequences, known as origins of replication, are recognized and bound by initiation proteins, and ends when the original DNA molecule has been completely duplicated and the copies topologically separated. The unit of replication usually corresponds to the genome of the cell, an organelle, or a virus. The template for replication can either be an existing DNA molecule or RNA.</td><td>IEA</td></tr><tr><td>GO:0030163</td><td>protein catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of a protein by the destruction of the native, active configuration, with or without the hydrolysis of peptide bonds.</td><td>IEA</td></tr><tr><td>GO:0016020</td><td>membrane</td><td>cellular_component</td><td>A lipid bilayer along with all the proteins and protein complexes embedded in it an attached to it.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G15200</li><li>BGIOSGA001752</li><li>OBART01G13840</li><li>ORGLA01G0115300</li><li>OMERI01G12210</li><li>AMDW01075951.1_FG002</li><li>OGLUM01G15450</li><li>OPUNC01G13470</li><li>LPERR01G12480</li><li>BRADI_2g12310v3</li><li>TraesCS1D01G067500</li><li>TRIUR3_04433</li><li>F775_28138</li><li>HORVU1Hr1G013740</li><li>TraesCS1A01G066400</li><li>TraesCS1B01G084700</li><li>SETIT_001653mg</li><li>SORBI_3003G152900</li><li>MANES_03G209100</li><li>TanjilG_29159</li><li>LR48_Vigan01g294900</li><li>PHAVU_003G118800g</li><li>TCM_021441</li><li>MTR_4g123070</li><li>Tp57577_TGAC_v2_gene12526</li><li>B456_005G069400</li><li>BVRB_3g059720</li><li>VIT_05s0020g00380</li><li>PRUPE_1G110900</li><li>CCACVL1_28645</li><li>CHLRE_18g749847v5</li><li>OSTLU_26614</li><li>CHC_T00010287001</li><li>Gasu_15110</li><li>CMM299C</li><li>ENSCSAVG00000000803</li><li>FBgn0036762</li><li>ENSG00000091140</li><li>WBGene00010794</li></ul></td></tr><tr><th>syntenic_ortholog_one2many</th><td><ul><li>Zm00001d009163</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os03g0163300</li><li>Os10g0415300</li><li>Os02g0813500</li><li>Os01g0337900</li><li>Os05g0156700</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0328700</td></tr><tr><th>Description</th><td>FAD-dependent pyridine nucleotide-disulphide oxidoreductase domain containing protein. (Os01t0328700-01)</td></tr><tr><th>Position</th><td>chr01:12655016..12658349</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td></td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr></table>