<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> ROS1A ROS1a DME1 </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> REPRESSOR OF SILENCING 1a </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Seed - Morphological traits - Endosperm Seed - Physiological traits - Dormancy </td></tr><tr><th>Gene Ontology</th><td> GO:0010162 - seed dormancy GO:0009960 - endosperm development GO:0009790 - embryonic development </td></tr><tr><th>Trait Ontology</th><td> TO:0000253 - seed dormancy TO:0000184 - seed anatomy and morphology trait </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0217900/Os01g0218032 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0218032</td></tr><tr><th>name</th><td>REPRESSOR OF SILENCING 1a</td></tr><tr><th>description</th><td>Putative DNA demethylase, Endosperm development (Os01t0218032-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>1783</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>6444246</td></tr><tr><th>end</th><td>6456068</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>RAP-DB</td><td><ul><li>ROS1A, ROS1a</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI000175BC68</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>ROS1A, ROS1a, DME1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR003265</td><td>HhH-GPD_domain</td><td>HhH-GPD domain</td></tr><tr><td>IPR003651</td><td>Endonuclease3_FeS-loop_motif</td><td>Endonuclease III-like, iron-sulphur cluster loop motif</td></tr><tr><td>IPR011257</td><td>DNA_glycosylase</td><td>DNA glycosylase</td></tr><tr><td>IPR023170</td><td>HTH_base_excis_C</td><td>Helix-turn-helix, base-excision DNA repair, C-terminal</td></tr><tr><td>IPR028924</td><td>Perm-CXXC</td><td>Permuted single zf-CXXC unit</td></tr><tr><td>IPR028925</td><td>RRM_DME</td><td>Demeter, RRM-fold domain</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0003824</td><td>catalytic activity</td><td>molecular_function</td><td>Catalysis of a biochemical reaction at physiological temperatures. In biologically catalyzed reactions, the reactants are known as substrates, and the catalysts are naturally occurring macromolecular substances known as enzymes. Enzymes possess specific binding sites for substrates, and are usually composed wholly or largely of protein, but RNA that has catalytic activity (ribozyme) is often also regarded as enzymatic.</td><td>IEA</td></tr><tr><td>GO:0051539</td><td>4 iron, 4 sulfur cluster binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a 4 iron, 4 sulfur (4Fe-4S) cluster; this cluster consists of four iron atoms, with the inorganic sulfur atoms found between the irons and acting as bridging ligands.</td><td>IEA</td></tr><tr><td>GO:0006284</td><td>base-excision repair</td><td>biological_process</td><td>In base excision repair, an altered base is removed by a DNA glycosylase enzyme, followed by excision of the resulting sugar phosphate. The small gap left in the DNA helix is filled in by the sequential action of DNA polymerase and DNA ligase.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G08030</li><li>BGIOSGA003012</li><li>OBART01G07400</li><li>ORGLA01G0065000</li><li>OGLUM01G08440</li><li>KN538689.1_FG117</li><li>OMERI01G07320</li><li>OPUNC01G07120</li><li>LPERR12G15400</li><li>OB09G11210</li><li>TraesCS5D01G173300</li><li>BRADI_4g08870v3</li><li>TraesCS5B01G165800</li><li>F775_06749</li><li>TraesCS5A01G169000</li><li>SETIT_028642mg</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os05g0445900</li><li>Os02g0496500</li><li>Os05g0446550</li><li>Os05g0446600</li><li>Os06g0237900</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>Dr00132</li><li>Dr11540</li><li>Dr13895</li><li>Solyc04g047820.1</li><li>Solyc04g054870.1</li><li>AMTR_s00088p00128580</li><li>AMTR_s00040p00215000</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsROS1|TA2'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0218032'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g11900'}</td></tr><tr><th>Keyword</th><td>{'map-based cloning'}</td></tr><tr><th>Title</th><td>{'Mutations in the DNA demethylase OsROS1 result in a thickened aleurone and improved nutritional value in rice grains.'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0218032</td></tr><tr><th>Description</th><td>Putative DNA demethylase, Endosperm development (Os01t0218032-01)</td></tr><tr><th>Position</th><td>chr01:6444246..6456068</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>ROS1A, ROS1a</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>REPRESSOR OF SILENCING 1a</td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>ROS1A, ROS1a, DME1</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>REPRESSOR OF SILENCING 1a</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsROS1|TA2'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0218032'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g11900'}</td></tr></table></td></tr></table>