<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> ATG18C </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsATG18c OsWD40-30 </td></tr><tr><th>CGSNL Gene Name</th><td> AUTOPHAGY ASSOCIATED GENE 18C </td></tr><tr><th>Gene name synonym(s)</th><td> autophagy 18c </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> </td></tr><tr><th>Gene Ontology</th><td> GO:0032266 - phosphatidylinositol 3-phosphate binding GO:0080025 - phosphatidylinositol-3,5-bisphosphate binding GO:0042594 - response to starvation GO:0031090 - organelle membrane GO:0006914 - autophagy GO:0000045 - autophagic vacuole formation GO:0005737 - cytoplasm GO:0010150 - leaf senescence </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0934000 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0934000</td></tr><tr><th>name</th><td>AUTOPHAGY ASSOCIATED GENE 18C</td></tr><tr><th>description</th><td>WD40 repeat-like domain containing protein. (Os01t0934000-01);WD40 repeat-like domain containing protein. (Os01t0934000-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>10392</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>40966688</td></tr><tr><th>end</th><td>40971055</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>ATG18C</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000AB46A</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q8LR61</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g70780.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015621196.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAF07221.1</li><li>BAS76065.1</li><li>EAZ14739.1</li><li>BAB93248.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK060679</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015765710.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326829</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsATG18c, OsWD40-30</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR001680</td><td>WD40_repeat</td><td>WD40 repeat</td></tr><tr><td>IPR017986</td><td>WD40_repeat_dom</td><td>WD40-repeat-containing domain</td></tr><tr><td>IPR036322</td><td>WD40_repeat_dom_sf</td><td>WD40-repeat-containing domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0005829</td><td>cytosol</td><td>cellular_component</td><td>The part of the cytoplasm that does not contain organelles but which does contain other particulate matter, such as protein complexes.</td><td>IBA</td></tr><tr><td>GO:0042594</td><td>response to starvation</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a starvation stimulus, deprivation of nourishment.</td><td>IEA</td></tr><tr><td>GO:0034497</td><td>protein localization to phagophore assembly site</td><td>biological_process</td><td>Any process in which a protein is transported to, or maintained at, the phagophore assembly site (PAS).</td><td>IBA</td></tr><tr><td>GO:0000422</td><td>autophagy of mitochondrion</td><td>biological_process</td><td>The autophagic process in which mitochondria are delivered to the vacuole and degraded in response to changing cellular conditions.</td><td>IBA</td></tr><tr><td>GO:0009651</td><td>response to salt stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating an increase or decrease in the concentration of salt (particularly but not exclusively sodium and chloride ions) in the environment.</td><td>IEA</td></tr><tr><td>GO:0044804</td><td>autophagy of nucleus</td><td>biological_process</td><td>A form of autophagy, by which damaged or non-essential parts of the nucleus, or even an entire nucleus is degraded.</td><td>IBA</td></tr><tr><td>GO:0034045</td><td>phagophore assembly site membrane</td><td>cellular_component</td><td>A cellular membrane associated with the phagophore assembly site.</td><td>IBA</td></tr><tr><td>GO:0050832</td><td>defense response to fungus</td><td>biological_process</td><td>Reactions triggered in response to the presence of a fungus that act to protect the cell or organism.</td><td>IEA</td></tr><tr><td>GO:0009414</td><td>response to water deprivation</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a water deprivation stimulus, prolonged deprivation of water.</td><td>IEA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IEA</td></tr><tr><td>GO:0019898</td><td>extrinsic component of membrane</td><td>cellular_component</td><td>The component of a membrane consisting of gene products and protein complexes that are loosely bound to one of its surfaces, but not integrated into the hydrophobic region.</td><td>IBA</td></tr><tr><td>GO:0010508</td><td>positive regulation of autophagy</td><td>biological_process</td><td>Any process that activates, maintains or increases the rate of autophagy. Autophagy is the process in which cells digest parts of their own cytoplasm.</td><td>IEA</td></tr><tr><td>GO:0010150</td><td>leaf senescence</td><td>biological_process</td><td>The process that occurs in a leaf near the end of its active life that is associated with the dismantling of cell components and membranes, loss of functional chloroplasts, and an overall decline in metabolism.</td><td>IEA</td></tr><tr><td>GO:0006497</td><td>protein lipidation</td><td>biological_process</td><td>The covalent attachment of lipid groups to an amino acid in a protein.</td><td>IBA</td></tr><tr><td>GO:0080025</td><td>phosphatidylinositol-3,5-bisphosphate binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with phosphatidylinositol-3,5-bisphosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' and 5' positions.</td><td>IBA</td></tr><tr><td>GO:0032266</td><td>phosphatidylinositol-3-phosphate binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with phosphatidylinositol-3-phosphate, a derivative of phosphatidylinositol in which the inositol ring is phosphorylated at the 3' position.</td><td>IBA</td></tr><tr><td>GO:0006979</td><td>response to oxidative stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of oxidative stress, a state often resulting from exposure to high levels of reactive oxygen species, e.g. superoxide anions, hydrogen peroxide (H2O2), and hydroxyl radicals.</td><td>IEA</td></tr><tr><td>GO:0005515</td><td>protein binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G46450</li><li>BGIOSGA000177</li><li>ORGLA01G0367600</li><li>OBART01G43020</li><li>KN538929.1_FG002</li><li>OPUNC01G41970</li><li>OB01G52030</li><li>LPERR01G37450</li><li>TRIUR3_04812</li><li>TraesCS3A01G453200</li><li>TraesCS3B01G490900</li><li>BRADI_2g59880v3</li><li>HORVU3Hr1G098020</li><li>F775_31100</li><li>SORBI_3003G416400</li><li>SETIT_001428mg</li><li>TanjilG_17603</li><li>PHAVU_007G196400g</li><li>HannXRQ_Chr16g0499791</li><li>PRUPE_6G322300</li><li>PGSC0003DMG400024704</li><li>MTR_1g083230</li><li>Solyc01g099400.2</li><li>Tp57577_TGAC_v2_gene6103</li><li>A4A49_28606</li><li>CCACVL1_13746</li><li>LR48_Vigan02g167400</li><li>VIT_14s0060g02380</li><li>Dr06999</li><li>AMTR_s00057p00210380</li></ul></td></tr><tr><th>syntenic_ortholog_one2many</th><td><ul><li>GSMUA_Achr8G10070_001</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os01g0168500</li><li>Os05g0169200</li><li>Os02g0791800</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>SELMODRAFT_177001</li><li>SELMODRAFT_171375</li><li>PHYPA_009501</li><li>PHYPA_021658</li><li>PHYPA_007298</li><li>ENSG00000196998</li><li>ENSG00000141580</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_faminfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'ATG18C'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0934000'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g70780'}</td></tr><tr><th>Name</th><td>{'ATG'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0934000</td></tr><tr><th>Description</th><td>WD40 repeat-like domain containing protein. (Os01t0934000-01);WD40 repeat-like domain containing protein. (Os01t0934000-02)</td></tr><tr><th>Position</th><td>chr01:40966688..40971055</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>ATG18C</td></tr><tr><th>CGSNL Gene Name</th><td>AUTOPHAGY ASSOCIATED GENE 18C</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsATG18c, OsWD40-30</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>autophagy 18c</td></tr></table></td></tr></table>