<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsMUS81 OsMUS81alpha OsMUS81beta ERCC4 </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0000724 - double-strand break repair via homologous recombination GO:0051026 - chiasma formation GO:0003677 - DNA binding GO:0000794 - condensed nuclear chromosome GO:0006312 - mitotic recombination GO:0004519 - endonuclease activity GO:0046872 - metal ion binding GO:0051301 - cell division GO:0007067 - mitosis </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0948100 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0948100</td></tr><tr><th>name</th><td>OsMUS81, OsMUS81alpha, OsMUS81beta, ERCC4</td></tr><tr><th>description</th><td>Restriction endonuclease, type II-like, core domain containing protein. (Os01t0948100-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>10613</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>41691190</td></tr><tr><th>end</th><td>41698689</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>Oryzabase</td><td><ul><li>OsMUS81, OsMUS81alpha, OsMUS81beta, ERCC4</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000AE3D5</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q8GT06</li></ul></td></tr><tr><td>Uniprot/Varsplic</td><td><ul><li>Q8GT06-1</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g71960.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015622535.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAC15787.1</li><li>BAS76212.1</li><li>BAF07304.1</li><li>EEE55999.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK111411</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015767049.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4325837</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR006166</td><td>ERCC4_domain</td><td>ERCC4 domain</td></tr><tr><td>IPR011335</td><td>Restrct_endonuc-II-like</td><td>Restriction endonuclease  type II-like</td></tr><tr><td>IPR036388</td><td>WH-like_DNA-bd_sf</td><td>Winged helix-like DNA-binding domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0051301</td><td>cell division</td><td>biological_process</td><td>The process resulting in division and partitioning of components of a cell to form more cells; may or may not be accompanied by the physical separation of a cell into distinct, individually membrane-bounded daughter cells.</td><td>IEA</td></tr><tr><td>GO:0031297</td><td>replication fork processing</td><td>biological_process</td><td>The process in which a DNA replication fork that has stalled is restored to a functional state and replication is restarted. The stalling may be due to DNA damage, DNA secondary structure, bound proteins, dNTP shortage, or other causes.</td><td>IBA</td></tr><tr><td>GO:0048257</td><td>3'-flap endonuclease activity</td><td>molecular_function</td><td>Catalysis of the cleavage of a 3' flap structure in DNA, but not other DNA structures; processes the 3' ends of Okazaki fragments in lagging strand DNA synthesis.</td><td>IBA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IBA</td></tr><tr><td>GO:0046872</td><td>metal ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any metal ion.</td><td>IEA</td></tr><tr><td>GO:0000712</td><td>resolution of meiotic recombination intermediates</td><td>biological_process</td><td>The cleavage and rejoining of intermediates, such as Holliday junctions, formed during meiotic recombination to produce two intact molecules in which genetic material has been exchanged.</td><td>IBA</td></tr><tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>Any molecular function by which a gene product interacts selectively and non-covalently with DNA (deoxyribonucleic acid).</td><td>IEA</td></tr><tr><td>GO:0007095</td><td>mitotic G2 DNA damage checkpoint</td><td>biological_process</td><td>A mitotic cell cycle checkpoint that detects and negatively regulates progression through the G2/M transition of the cell cycle in response to DNA damage.</td><td>IBA</td></tr><tr><td>GO:0048476</td><td>Holliday junction resolvase complex</td><td>cellular_component</td><td>A protein complex that mediates the conversion of a Holliday junction into two separate duplex DNA molecules; the complex includes a single- or multisubunit helicase that catalyzes the extension of heteroduplex DNA by branch migration and a nuclease that resolves the junction by nucleolytic cleavage.</td><td>IBA</td></tr><tr><td>GO:0031573</td><td>intra-S DNA damage checkpoint</td><td>biological_process</td><td>A mitotic cell cycle checkpoint that slows DNA synthesis in response to DNA damage by the prevention of new origin firing and the stabilization of slow replication fork progression.</td><td>IBA</td></tr><tr><td>GO:0000727</td><td>double-strand break repair via break-induced replication</td><td>biological_process</td><td>The error-free repair of a double-strand break in DNA in which the centromere-proximal end of a broken chromosome searches for a homologous region in an intact chromosome. DNA synthesis initiates from the 3' end of the invading DNA strand, using the intact chromosome as the template, and progresses to the end of the chromosome.</td><td>IBA</td></tr><tr><td>GO:0008821</td><td>crossover junction endodeoxyribonuclease activity</td><td>molecular_function</td><td>Catalysis of the endonucleolytic cleavage at a junction such as a reciprocal single-stranded crossover between two homologous DNA duplexes (Holliday junction).</td><td>IBA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G47290</li><li>BGIOSGA005153</li><li>AMDW01032941.1_FG001</li><li>OPUNC01G42750</li><li>OB01G52880</li><li>LPERR01G38140</li><li>CMI043C</li><li>OSTLU_92534</li><li>MTR_3g022850</li><li>Bra029467</li><li>MANES_04G164000</li><li>HannXRQ_Chr05g0145061</li><li>DCAR_009210</li><li>TCM_042390</li><li>CCACVL1_30726</li><li>B456_013G015500</li><li>Csa_5G623550</li><li>GLYMA_11G150100</li><li>PHAVU_005G033000g</li><li>Tp57577_TGAC_v2_gene18800</li><li>Bo9g078480</li><li>VIT_13s0019g01290</li><li>LR48_Vigan05g013400</li><li>BVRB_2g047480</li><li>POPTR_003G202300v3</li><li>Dr09079</li><li>PHYPA_020862</li><li>CHLRE_12g555050v5</li><li>Gasu_19150</li><li>WBGene00016602</li><li>YDR386W</li><li>ENSG00000172732</li><li>ENSCSAVG00000002776</li><li>FBgn0040347</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0948100</td></tr><tr><th>Description</th><td>Restriction endonuclease, type II-like, core domain containing protein. (Os01t0948100-01)</td></tr><tr><th>Position</th><td>chr01:41691190..41698689</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsMUS81, OsMUS81alpha, OsMUS81beta, ERCC4</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsMUS81'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0948100'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g71960'}</td></tr></table></td></tr></table>