<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> P5CR </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsP5CR </td></tr><tr><th>CGSNL Gene Name</th><td> PYRROLINE-5-CARBOXYLATE REDUCTASE </td></tr><tr><th>Gene name synonym(s)</th><td> delta-pyrroline-5-carboxylate reductase pyrroline-5-carboxylate reductase </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character Tolerance and resistance - Stress tolerance </td></tr><tr><th>Gene Ontology</th><td> GO:0004735 - pyrroline-5-carboxylate reductase activity GO:0005618 - cell wall GO:0006561 - proline biosynthetic process GO:0006970 - response to osmotic stress GO:0009408 - response to heat GO:0009651 - response to salt stress </td></tr><tr><th>Trait Ontology</th><td> TO:0000095 - osmotic response sensitivity TO:0006002 - proline content </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0948400 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0948400</td></tr><tr><th>name</th><td>PYRROLINE-5-CARBOXYLATE REDUCTASE</td></tr><tr><th>description</th><td>Similar to Pyrroline-5-carboxylate reductase (EC 1.5.1.2) (P5CR) (P5C reductase). (Os01t0948400-01);Similar to pyrroline 5-carboxylate reductase. (Os01t0948400-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>10624</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>41718639</td></tr><tr><th>end</th><td>41721571</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>P5CR</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0000DD8EC3</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q0JG21</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAF07307.1</li><li>BAS76215.1</li></ul></td></tr><tr><td>KEGG_Enzyme</td><td><ul><li>00330+1.5.1.2</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4325755</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsP5CR</li></ul></td></tr><tr><td>notGramene_Plant_Reactome</td><td><ul><li>R-OSA-1119631.1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>pathways</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ancestors</th><td><ul><li>2744345</li><li>2744343</li><li>5655122</li><li>1119631</li><li>1119318</li><li>1119289</li></ul></td></tr><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>link</th><th>name</th></tr><tr><td>R-OSA-1121255</td><td>http://plantreactome.gramene.org/PathwayBrowser/#/R-OSA-1121255</td><td>pyrroline-5-carboxylate reductase</td></tr><tr><td>R-OSA-5631956</td><td>http://plantreactome.gramene.org/PathwayBrowser/#/R-OSA-5631956</td><td>Conversion of P5C to L-Pro</td></tr></table></td></tr></table></td></tr><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR000304</td><td>Pyrroline-COOH_reductase</td><td>Pyrroline-5-carboxylate reductase</td></tr><tr><td>IPR008927</td><td>6-PGluconate_DH-like_C_sf</td><td>6-phosphogluconate dehydrogenase-like, C-terminal domain superfamily</td></tr><tr><td>IPR028939</td><td>P5C_Rdtase_cat_N</td><td>Pyrroline-5-carboxylate reductase, catalytic, N-terminal</td></tr><tr><td>IPR029036</td><td>P5CR_dimer</td><td>Pyrroline-5-carboxylate reductase, dimerisation domain</td></tr><tr><td>IPR036291</td><td>NAD(P)-bd_dom_sf</td><td>NAD(P)-binding domain superfamily</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>16040</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0005618</td><td>cell wall</td><td>cellular_component</td><td>The rigid or semi-rigid envelope lying outside the cell membrane of plant, fungal, most prokaryotic cells and some protozoan parasites, maintaining their shape and protecting them from osmotic lysis. In plants it is made of cellulose and, often, lignin; in fungi it is composed largely of polysaccharides; in bacteria it is composed of peptidoglycan; in protozoan parasites such as Giardia species, it's made of carbohydrates and proteins.</td><td>IEA</td></tr><tr><td>GO:0004735</td><td>pyrroline-5-carboxylate reductase activity</td><td>molecular_function</td><td>Catalysis of the reaction: L-proline + NADP+ = 1-pyrroline-5-carboxylate + NADPH + H+.</td><td>IEA</td></tr><tr><td>GO:0055129</td><td>L-proline biosynthetic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the formation of L-proline, an L-enantiomer of a chiral, cyclic, nonessential alpha-amino acid found in peptide linkage in proteins.</td><td>IEA</td></tr><tr><td>GO:0009651</td><td>response to salt stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating an increase or decrease in the concentration of salt (particularly but not exclusively sodium and chloride ions) in the environment.</td><td>IEA</td></tr><tr><td>GO:0009408</td><td>response to heat</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a heat stimulus, a temperature stimulus above the optimal temperature for that organism.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G47310</li><li>BGIOSGA000114</li><li>ORGLA01G0376900</li><li>KN539271.1_FG011</li><li>OPUNC01G42800</li><li>LPERR01G38160</li><li>BRADI_2g60730v3</li><li>HORVU3Hr1G106720</li><li>TraesCS3D01G483400</li><li>F775_09421</li><li>TraesCS3B01G538100</li><li>SETIT_002357mg</li><li>Zm00001d042117</li><li>SORBI_3003G424300</li><li>TRIUR3_26669</li><li>GSMUA_Achr11G20240_001</li><li>Dr07149</li><li>Csa_4G354630</li><li>CCACVL1_25558</li><li>AT5G14800</li><li>PHAVU_001G124300g</li><li>BVRB_3g062510</li><li>LR48_Vigan03g209600</li><li>Tp57577_TGAC_v2_gene9312</li><li>Solyc02g068640.2</li><li>A4A49_21201</li><li>Al_scaffold_0006_1383</li><li>KK1_049850</li><li>MTR_7g090160</li><li>PGSC0003DMG400010441</li><li>DCAR_020545</li><li>B456_001G029600</li><li>TCM_024404</li><li>VIT_08s0007g05600</li><li>POPTR_006G042600v3</li><li>HannXRQ_Chr09g0266491</li><li>AMTR_s00053p00048140</li><li>PHYPA_013925</li><li>CHLRE_03g154550v5</li><li>OSTLU_13156</li><li>WBGene00010924</li><li>CMI158C</li><li>CHC_T00001376001</li><li>YER023W</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsP5CR'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0948400'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g71990'}</td></tr><tr><th>Keyword</th><td>{'resistant', 'salt tolerance', 'salt stress', 'salt', ' ABA '}</td></tr><tr><th>Title</th><td>{'Exogenous ABA induces salt tolerance in indica rice (Oryza sativa L.): The role of OsP5CS1 and OsP5CR gene expression during salt stress'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0948400</td></tr><tr><th>Description</th><td>Similar to Pyrroline-5-carboxylate reductase (EC 1.5.1.2) (P5CR) (P5C reductase). (Os01t0948400-01);Similar to pyrroline 5-carboxylate reductase. (Os01t0948400-02)</td></tr><tr><th>Position</th><td>chr01:41718639..41721571</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>P5CR</td></tr><tr><th>CGSNL Gene Name</th><td>PYRROLINE-5-CARBOXYLATE REDUCTASE</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsP5CR</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>delta-pyrroline-5-carboxylate reductase, pyrroline-5-carboxylate reductase</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsP5CR'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0948400'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g71990'}</td></tr></table></td></tr></table>