<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Other </td></tr><tr><th>Gene Ontology</th><td> GO:0003712 - transcription cofactor activity GO:0008270 - zinc ion binding GO:0009553 - embryo sac development GO:0010227 - floral organ abscission GO:0005634 - nucleus GO:0006355 - regulation of transcription, DNA-dependent GO:0009954 - proximal/distal pattern formation GO:0009555 - pollen development GO:0048439 - flower morphogenesis GO:0048573 - photoperiodism, flowering GO:0004402 - histone acetyltransferase activity </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0893400 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0893400</td></tr><tr><th>name</th><td>Os01g0893400</td></tr><tr><th>description</th><td>Zinc finger, TAZ-type domain containing protein. (Os01t0893400-01);Zinc finger, TAZ-type domain containing protein. (Os01t0893400-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9779</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>38843729</td></tr><tr><th>end</th><td>38846523</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>UniParc</td><td><ul><li>UPI00004C29EB</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q5JLW2</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g66890.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015640130.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>EEE55796.1</li><li>BAS75678.1</li><li>BAF06978.1</li><li>BAD87545.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK071112</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015784644.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4324958</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR000197</td><td>Znf_TAZ</td><td>Zinc finger, TAZ-type</td></tr><tr><td>IPR000210</td><td>BTB/POZ_dom</td><td>BTB/POZ domain</td></tr><tr><td>IPR011333</td><td>SKP1/BTB/POZ_sf</td><td>SKP1/BTB/POZ domain superfamily</td></tr><tr><td>IPR034089</td><td>SPOP_C</td><td>SPOP, C-terminal BACK-like domain</td></tr><tr><td>IPR035898</td><td>TAZ_dom_sf</td><td>TAZ domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0019005</td><td>SCF ubiquitin ligase complex</td><td>cellular_component</td><td>A ubiquitin ligase complex in which a cullin from the Cul1 subfamily and a RING domain protein form the catalytic core; substrate specificity is conferred by a Skp1 adaptor and an F-box protein. SCF complexes are involved in targeting proteins for degradation by the proteasome. The best characterized complexes are those from yeast and mammals (with core subunits named Cdc53/Cul1, Rbx1/Hrt1/Roc1).</td><td>IBA</td></tr><tr><td>GO:0043161</td><td>proteasome-mediated ubiquitin-dependent protein catabolic process</td><td>biological_process</td><td>The chemical reactions and pathways resulting in the breakdown of a protein or peptide by hydrolysis of its peptide bonds, initiated by the covalent attachment of ubiquitin, and mediated by the proteasome.</td><td>IBA</td></tr><tr><td>GO:0009723</td><td>response to ethylene</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ethylene (ethene) stimulus.</td><td>IEA</td></tr><tr><td>GO:0005737</td><td>cytoplasm</td><td>cellular_component</td><td>All of the contents of a cell excluding the plasma membrane and nucleus, but including other subcellular structures.</td><td>IBA</td></tr><tr><td>GO:0003712</td><td>transcription coregulator activity</td><td>molecular_function</td><td>A protein or a member of a complex that interacts specifically and non-covalently with a DNA-binding transcription factor to either activate or repress the transcription of specific genes. Coregulators often act by altering chromatin structure and modifications. For example, one class of transcription coregulators modifies chromatin structure through covalent modification of histones. A second ATP-dependent class modifies the conformation of chromatin. Another type of coregulator activity is the bridging of a DNA-binding transcription factor to the basal transcription machinery. The Mediator complex, which bridges transcription factors and RNA polymerase, is also a transcription coregulator (specifically, a coactivator).</td><td>IEA</td></tr><tr><td>GO:0009555</td><td>pollen development</td><td>biological_process</td><td>The process whose specific outcome is the progression of the pollen grain over time, from its formation to the mature structure. The process begins with the meiosis of the microsporocyte to form four haploid microspores. The nucleus of each microspore then divides by mitosis to form a two-celled organism, the pollen grain, that contains a tube cell as well as a smaller generative cell. The pollen grain is surrounded by an elaborate cell wall. In some species, the generative cell immediately divides again to give a pair of sperm cells. In most flowering plants, however this division takes place later, in the tube that develops when a pollen grain germinates.</td><td>IEA</td></tr><tr><td>GO:0030162</td><td>regulation of proteolysis</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of the hydrolysis of a peptide bond or bonds within a protein.</td><td>IBA</td></tr><tr><td>GO:0009751</td><td>response to salicylic acid</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a salicylic acid stimulus.</td><td>IEA</td></tr><tr><td>GO:0042542</td><td>response to hydrogen peroxide</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a hydrogen peroxide (H2O2) stimulus.</td><td>IEA</td></tr><tr><td>GO:0031625</td><td>ubiquitin protein ligase binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a ubiquitin protein ligase enzyme, any of the E3 proteins.</td><td>IBA</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IEA</td></tr><tr><td>GO:0009651</td><td>response to salt stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating an increase or decrease in the concentration of salt (particularly but not exclusively sodium and chloride ions) in the environment.</td><td>IEA</td></tr><tr><td>GO:0009409</td><td>response to cold</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a cold stimulus, a temperature stimulus below the optimal temperature for that organism.</td><td>IEA</td></tr><tr><td>GO:0004402</td><td>histone acetyltransferase activity</td><td>molecular_function</td><td>Catalysis of the reaction: acetyl-CoA + histone = CoA + acetyl-histone.</td><td>IEA</td></tr><tr><td>GO:0006355</td><td>regulation of transcription, DNA-templated</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of cellular DNA-templated transcription.</td><td>IEA</td></tr><tr><td>GO:0005516</td><td>calmodulin binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with calmodulin, a calcium-binding protein with many roles, both in the calcium-bound and calcium-free states.</td><td>IEA</td></tr><tr><td>GO:0008270</td><td>zinc ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with zinc (Zn) ions.</td><td>IEA</td></tr><tr><td>GO:0009553</td><td>embryo sac development</td><td>biological_process</td><td>The process whose specific outcome is the progression of the embryo sac over time, from its formation to the mature structure. The process begins with the meiosis of the megasporocyte to form four haploid megaspores. Three of the megaspores disintegrate, and the fourth undergoes mitosis giving rise to a binucleate syncytial embryo sac. The two haploid nuclei migrate to the opposite poles of the embryo sac and then undergo two rounds of mitosis generating four haploid nuclei at each pole. One nucleus from each set of four migrates to the center of the cell. Cellularization occurs, resulting in an eight-nucleate seven-celled structure. This structure contains two synergid cells and an egg cell at the micropylar end, and three antipodal cells at the other end. A binucleate endosperm mother cell is formed at the center. The two polar nuclei fuse resulting in a mononucleate diploid endosperm mother cell. The three antipodal cells degenerate.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G43500</li><li>ORGLA01G0341200</li><li>OBART01G40170</li><li>OMERI01G37250</li><li>KN540649.1_FG009</li><li>BGIOSGA004968</li><li>ONIVA01G45220</li><li>OPUNC01G38850</li><li>OB01G48870</li><li>LPERR01G34680</li><li>TraesCS3A01G400400</li><li>HORVU3Hr1G090000</li><li>BRADI_2g57476v3</li><li>TraesCS3B01G433400</li><li>SETIT_001548mg</li><li>SORBI_3003G385100</li><li>GSMUA_Achr6G18360_001</li><li>BVRB_3g058930</li><li>PRUPE_7G267800</li><li>A4A49_03095</li><li>TCM_019806</li><li>CCACVL1_18339</li><li>AT1G05690</li><li>MANES_S012900</li><li>VIT_05s0094g00950</li><li>scaffold_100554.1</li><li>DCAR_023778</li><li>Bra030621</li><li>AMTR_s00066p00175680</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os04g0482300</li><li>Os01g0908200</li><li>Os02g0596700</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>SELMODRAFT_13821</li><li>SELMODRAFT_13820</li><li>PHYPA_020045</li><li>PHYPA_029222</li><li>PHYPA_012980</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0893400</td></tr><tr><th>Description</th><td>Zinc finger, TAZ-type domain containing protein. (Os01t0893400-01);Zinc finger, TAZ-type domain containing protein. (Os01t0893400-02)</td></tr><tr><th>Position</th><td>chr01:38843729..38846523</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td></td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr></table>