<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> CDC6 </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsCDC6 </td></tr><tr><th>CGSNL Gene Name</th><td> CELL DIVISION CONTROL 6 </td></tr><tr><th>Gene name synonym(s)</th><td> </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Biochemical character </td></tr><tr><th>Gene Ontology</th><td> GO:0005634 - nucleus GO:0033314 - mitotic cell cycle DNA replication checkpoint GO:0000278 - mitotic cell cycle GO:0006270 - DNA replication initiation GO:0051301 - cell division GO:0003688 - DNA replication origin binding </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0856000 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0856000</td></tr><tr><th>name</th><td>CELL DIVISION CONTROL 6</td></tr><tr><th>description</th><td>Similar to CDC6 protein. (Os01t0856000-01);Similar to predicted protein. (Os01t0856000-02)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>9245</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>36952073</td></tr><tr><th>end</th><td>36956727</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>CDC6</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI000049E9F8</li></ul></td></tr><tr><td>Uniprot/SWISSPROT</td><td><ul><li>Q5N897</li></ul></td></tr><tr><td>Uniprot/Varsplic</td><td><ul><li>Q5N897-1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS75292.1</li><li>BAS75291.1</li><li>BAF06759.2</li><li>BAD82304.1</li><li>BAD82303.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK107451</li><li>AK107439</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>OsCDC6</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR003593</td><td>AAA+_ATPase</td><td>AAA+ ATPase domain</td></tr><tr><td>IPR015163</td><td>Cdc6_C</td><td>Cdc6, C-terminal</td></tr><tr><td>IPR016314</td><td>Cdc6/18</td><td>Cell division protein Cdc6/18</td></tr><tr><td>IPR027417</td><td>P-loop_NTPase</td><td>P-loop containing nucleoside triphosphate hydrolase</td></tr><tr><td>IPR036388</td><td>WH-like_DNA-bd_sf</td><td>Winged helix-like DNA-binding domain superfamily</td></tr><tr><td>IPR036390</td><td>WH_DNA-bd_sf</td><td>Winged helix DNA-binding domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IEA</td></tr><tr><td>GO:0006270</td><td>DNA replication initiation</td><td>biological_process</td><td>The process in which DNA-dependent DNA replication is started; this begins with the ATP dependent loading of an initiator complex onto the DNA, this is followed by DNA melting and helicase activity. In bacteria, the gene products that enable the helicase activity are loaded after the initial melting and in archaea and eukaryotes, the gene products that enable the helicase activity are inactive when they are loaded and subsequently activate.</td><td>IEA</td></tr><tr><td>GO:0051301</td><td>cell division</td><td>biological_process</td><td>The process resulting in division and partitioning of components of a cell to form more cells; may or may not be accompanied by the physical separation of a cell into distinct, individually membrane-bounded daughter cells.</td><td>IEA</td></tr><tr><td>GO:0007049</td><td>cell cycle</td><td>biological_process</td><td>The progression of biochemical and morphological phases and events that occur in a cell during successive cell replication or nuclear replication events. Canonically, the cell cycle comprises the replication and segregation of genetic material followed by the division of the cell, but in endocycles or syncytial cells nuclear replication or nuclear division may not be followed by cell division.</td><td>IEA</td></tr><tr><td>GO:0003682</td><td>chromatin binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with chromatin, the network of fibers of DNA, protein, and sometimes RNA, that make up the chromosomes of the eukaryotic nucleus during interphase.</td><td>IEA</td></tr><tr><td>GO:0005524</td><td>ATP binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator.</td><td>IEA</td></tr><tr><td>GO:0008168</td><td>methyltransferase activity</td><td>molecular_function</td><td>Catalysis of the transfer of a methyl group to an acceptor molecule.</td><td>IEA</td></tr><tr><td>GO:0001510</td><td>RNA methylation</td><td>biological_process</td><td>Posttranscriptional addition of a methyl group to either a nucleotide or 2'-O ribose in a polyribonucleotide. Usually uses S-adenosylmethionine as a cofactor.</td><td>IEA</td></tr><tr><td>GO:0009452</td><td>7-methylguanosine RNA capping</td><td>biological_process</td><td>The sequence of enzymatic reactions by which the 5' cap structure, an inverted 7-methylguanosine linked via a 5'-5' triphosphate bridge (m7G(5')ppp(5')X) to the first transcribed residue, is added to a nascent transcript.</td><td>IEA</td></tr><tr><td>GO:0003677</td><td>DNA binding</td><td>molecular_function</td><td>Any molecular function by which a gene product interacts selectively and non-covalently with DNA (deoxyribonucleic acid).</td><td>IEA</td></tr><tr><td>GO:0046872</td><td>metal ion binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any metal ion.</td><td>IEA</td></tr><tr><td>GO:0006351</td><td>transcription, DNA-templated</td><td>biological_process</td><td>The cellular synthesis of RNA on a template of DNA.</td><td>IEA</td></tr><tr><td>GO:0006355</td><td>regulation of transcription, DNA-templated</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of cellular DNA-templated transcription.</td><td>IEA</td></tr><tr><td>GO:0009744</td><td>response to sucrose</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a sucrose stimulus.</td><td>IDA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ONIVA01G42520</li><li>ORGLA01G0315400</li><li>KN541504.1_FG002</li><li>OMERI01G34710</li><li>OPUNC01G36580</li><li>OB01G46480</li><li>BGIOSGA004825</li><li>OBART01G37740</li><li>ORUFI01G41070</li><li>LPERR01G32400</li><li>TraesCS3D01G362000</li><li>F775_01934</li><li>TraesCS3B01G400900</li><li>HORVU3Hr1G084800</li><li>TraesCS3A01G369000</li><li>BRADI_2g55320v3</li><li>TRIUR3_27338</li><li>SORBI_3003G361200</li><li>TanjilG_32180</li><li>CCACVL1_10904</li><li>PRUPE_6G059500</li><li>DCAR_018959</li><li>Bra015569</li><li>PHAVU_006G064300g</li><li>Solyc06g076860.2</li><li>BVRB_7g172450</li><li>B456_003G123500</li><li>LR48_Vigan09g116000</li><li>MANES_01G163000</li><li>PGSC0003DMG400030351</li><li>Csa_6G186380</li><li>MTR_3g463490</li><li>AMTR_s00029p00230900</li><li>PHYPA_002659</li><li>OSTLU_37384</li><li>YJL194W</li><li>ENSCSAVG00000001779</li><li>FBgn0035918</li><li>ENSG00000094804</li><li>CME082C</li><li>WBGene00000382</li><li>CHLRE_06g292850v5</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os06g0187100</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_faminfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'CDC6'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0856000'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g63710'}</td></tr><tr><th>Name</th><td>{'CDC'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0856000</td></tr><tr><th>Description</th><td>Similar to CDC6 protein. (Os01t0856000-01);Similar to predicted protein. (Os01t0856000-02)</td></tr><tr><th>Position</th><td>chr01:36952073..36956727</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td>CDC6</td></tr><tr><th>CGSNL Gene Name</th><td>CELL DIVISION CONTROL 6</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsCDC6</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr></table>