<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> _ </td></tr><tr><th>Gene symbol synonym(s)</th><td> OsRH58 </td></tr><tr><th>CGSNL Gene Name</th><td> _ </td></tr><tr><th>Gene name synonym(s)</th><td> RNA helicase 58 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> </td></tr><tr><th>Gene Ontology</th><td> GO:0009507 - chloroplast GO:0008026 - ATP-dependent helicase activity GO:0005524 - ATP binding GO:0003723 - RNA binding </td></tr><tr><th>Trait Ontology</th><td> </td></tr><tr><th>Plant Ontology</th><td> </td></tr><tr><th>RAP ID</th><td> Os01g0970600 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0970600</td></tr><tr><th>name</th><td>RNA helicase 58</td></tr><tr><th>description</th><td>Similar to ATP binding protein. (Os01t0970600-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>10973</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>42821254</td></tr><tr><th>end</th><td>42825043</td></tr><tr><th>strand</th><td>-1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>Oryzabase</td><td><ul><li>OsRH58</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI0003939A15</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>A0A0P0VDB4</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS76421.1</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR001650</td><td>Helicase_C</td><td>Helicase, C-terminal</td></tr><tr><td>IPR011545</td><td>DEAD/DEAH_box_helicase_dom</td><td>DEAD/DEAH box helicase domain</td></tr><tr><td>IPR014014</td><td>RNA_helicase_DEAD_Q_motif</td><td>RNA helicase, DEAD-box type, Q motif</td></tr><tr><td>IPR027417</td><td>P-loop_NTPase</td><td>P-loop containing nucleoside triphosphate hydrolase</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>14001</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0005524</td><td>ATP binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with ATP, adenosine 5'-triphosphate, a universally important coenzyme and enzyme regulator.</td><td>IEA</td></tr><tr><td>GO:0003723</td><td>RNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with an RNA molecule or a portion thereof.</td><td>IEA</td></tr><tr><td>GO:0004004</td><td>ATP-dependent RNA helicase activity</td><td>molecular_function</td><td>Catalysis of the reaction: ATP + H2O = ADP + phosphate; this reaction drives the unwinding of an RNA helix.</td><td>IBA</td></tr><tr><td>GO:0010501</td><td>RNA secondary structure unwinding</td><td>biological_process</td><td>The process in which a secondary structure of RNA are broken or 'melted'.</td><td>IBA</td></tr><tr><td>GO:0005730</td><td>nucleolus</td><td>cellular_component</td><td>A small, dense body one or more of which are present in the nucleus of eukaryotic cells. It is rich in RNA and protein, is not bounded by a limiting membrane, and is not seen during mitosis. Its prime function is the transcription of the nucleolar DNA into 45S ribosomal-precursor RNA, the processing of this RNA into 5.8S, 18S, and 28S components of ribosomal RNA, and the association of these components with 5S RNA and proteins synthesized outside the nucleolus. This association results in the formation of ribonucleoprotein precursors; these pass into the cytoplasm and mature into the 40S and 60S subunits of the ribosome.</td><td>IBA</td></tr><tr><td>GO:0009507</td><td>chloroplast</td><td>cellular_component</td><td>A chlorophyll-containing plastid with thylakoids organized into grana and frets, or stroma thylakoids, and embedded in a stroma.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G48840</li><li>BGIOSGA000030</li><li>OBART01G45320</li><li>ORGLA01G0391100</li><li>KN538778.1_FG032</li><li>OPUNC01G44270</li><li>OB01G54520</li><li>LPERR01G39760</li><li>TRIUR3_01589</li><li>BRADI_2g61960v3</li><li>HORVU3Hr1G114220</li><li>TraesCS3B01G592900</li><li>TraesCS3D01G530600</li><li>SORBI_3003G442700</li><li>Zm00001d011790</li><li>SETIT_017165mg</li><li>GSMUA_Achr6G27990_001</li><li>Dr17088</li><li>PGSC0003DMG400008995</li><li>Bra023712</li><li>Csa_3G149880</li><li>MTR_4g131750</li><li>TCM_020550</li><li>CCACVL1_23650</li><li>Tp57577_TGAC_v2_gene37009</li><li>fgenesh2_kg.6__1940__AT5G19210.2</li><li>HannXRQ_Chr17g0534161</li><li>LR48_Vigan01g338600</li><li>DCAR_027172</li><li>Bo2g017790</li><li>BVRB_004370</li><li>MANES_03G182200</li><li>B456_011G165900</li><li>VIT_05s0020g02870</li><li>AT5G19210</li><li>PRUPE_1G153600</li><li>POPTR_010G092200v3</li><li>Solyc01g079800.2</li><li>AMTR_s00131p00060240</li><li>PHYPA_009079</li><li>Gasu_41750</li><li>CMR374C</li><li>CHC_T00001396001</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os03g0219700</li><li>Os02g0636300</li><li>Os10g0503700</li><li>Os02g0641800</li><li>Os01g0549700</li><li>Os03g0158200</li><li>Os02g0146600</li><li>Os06g0701100</li><li>Os01g0550000</li><li>Os04g0533000</li><li>Os01g0639100</li><li>Os03g0566800</li><li>Os02g0221300</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsRH58'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0970600'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g73900'}</td></tr><tr><th>Keyword</th><td>{'abiotic stress', 'chloroplast', 'biotic stress', 'tolerance', 'ABA', 'stress', 'stress tolerance', ' ABA '}</td></tr><tr><th>Title</th><td>{'Rice OsRH58, a chloroplast DEAD-box RNA helicase, improves salt or drought stress tolerance in Arabidopsis by affecting chloroplast translation.'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0970600</td></tr><tr><th>Description</th><td>Similar to ATP binding protein. (Os01t0970600-01)</td></tr><tr><th>Position</th><td>chr01:42821254..42825043</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>OsRH58</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>RNA helicase 58</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'OsRH58'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0970600'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g73900'}</td></tr></table></td></tr></table>