<table style="width:100%" class="table table-striped" border="1"><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0337900</td></tr><tr><th>name</th><td>Os01g0337900</td></tr><tr><th>description</th><td>Similar to Dihydrolipoamide dehydrogenase. (Os01t0337900-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>3461</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>13272614</td></tr><tr><th>end</th><td>13282008</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>UniParc</td><td><ul><li>UPI000009D3C3</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q94CN9</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g23610.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015625013.1</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAB63876.1</li><li>BAF04855.1</li><li>BAS71953.1</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK120645</li></ul></td></tr><tr><td>KEGG_Enzyme</td><td><ul><li>00260+1.8.1.4</li><li>00640+1.8.1.4</li><li>00620+1.8.1.4</li><li>00020+1.8.1.4</li><li>00010+1.8.1.4</li><li>00280+1.8.1.4</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015769527.1</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4326849</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR004099</td><td>Pyr_nucl-diS_OxRdtase_dimer</td><td>Pyridine nucleotide-disulphide oxidoreductase, dimerisation domain</td></tr><tr><td>IPR006258</td><td>Lipoamide_DH</td><td>Dihydrolipoamide dehydrogenase</td></tr><tr><td>IPR012999</td><td>Pyr_OxRdtase_I_AS</td><td>Pyridine nucleotide-disulphide oxidoreductase, class I, active site</td></tr><tr><td>IPR016156</td><td>FAD/NAD-linked_Rdtase_dimer_sf</td><td>FAD/NAD-linked reductase, dimerisation domain superfamily</td></tr><tr><td>IPR023753</td><td>FAD/NAD-binding_dom</td><td>FAD/NAD(P)-binding domain</td></tr><tr><td>IPR036188</td><td>FAD/NAD-bd_sf</td><td>FAD/NAD(P)-binding domain superfamily</td></tr></table></td></tr><tr><th>ancestors</th><td><ul><li>1100</li></ul></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0004148</td><td>dihydrolipoyl dehydrogenase activity</td><td>molecular_function</td><td>Catalysis of the reaction: protein N6-(dihydrolipoyl)lysine + NAD+ = protein N6-(lipoyl)lysine + NADH + H+.</td><td>IEA</td></tr><tr><td>GO:0050660</td><td>flavin adenine dinucleotide binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with FAD, flavin-adenine dinucleotide, the coenzyme or the prosthetic group of various flavoprotein oxidoreductase enzymes, in either the oxidized form, FAD, or the reduced form, FADH2.</td><td>IEA</td></tr><tr><td>GO:0009055</td><td>electron transfer activity</td><td>molecular_function</td><td>Any molecular entity that serves as an electron acceptor and electron donor in an electron transport chain. An electron transport chain is a process in which a series of electron carriers operate together to transfer electrons from donors to any of several different terminal electron acceptors to generate a transmembrane electrochemical gradient.</td><td>IEA</td></tr><tr><td>GO:0045454</td><td>cell redox homeostasis</td><td>biological_process</td><td>Any process that maintains the redox environment of a cell or compartment within a cell.</td><td>IEA</td></tr><tr><td>GO:0003924</td><td>GTPase activity</td><td>molecular_function</td><td>Catalysis of the reaction: GTP + H2O = GDP + phosphate.</td><td>IEA</td></tr><tr><td>GO:0005525</td><td>GTP binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with GTP, guanosine triphosphate.</td><td>IEA</td></tr><tr><td>GO:0007020</td><td>microtubule nucleation</td><td>biological_process</td><td>The process in which tubulin alpha-beta heterodimers begin aggregation to form an oligomeric tubulin structure (a microtubule seed). Microtubule nucleation is the initiating step in the formation of a microtubule in the absence of any existing microtubules ('de novo' microtubule formation).</td><td>IEA</td></tr><tr><td>GO:0031122</td><td>cytoplasmic microtubule organization</td><td>biological_process</td><td>A process that is carried out at the cellular level which results in the assembly, arrangement of constituent parts, or disassembly of structures formed of microtubules and associated proteins in the cytoplasm of a cell.</td><td>IEA</td></tr><tr><td>GO:0000930</td><td>gamma-tubulin complex</td><td>cellular_component</td><td>A multiprotein complex composed of gamma-tubulin and other non-tubulin proteins. Gamma-tubulin complexes are localized to microtubule organizing centers, and play an important role in the nucleation of microtubules. The number and complexity of non-tubulin proteins associated with these complexes varies between species.</td><td>IEA</td></tr><tr><td>GO:0005874</td><td>microtubule</td><td>cellular_component</td><td>Any of the long, generally straight, hollow tubes of internal diameter 12-15 nm and external diameter 24 nm found in a wide variety of eukaryotic cells; each consists (usually) of 13 protofilaments of polymeric tubulin, staggered in such a manner that the tubulin monomers are arranged in a helical pattern on the microtubular surface, and with the alpha/beta axes of the tubulin subunits parallel to the long axis of the tubule; exist in equilibrium with pool of tubulin monomers and can be rapidly assembled or disassembled in response to physiological stimuli; concerned with force generation, e.g. in the spindle.</td><td>IEA</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G15690</li><li>BGIOSGA003432</li><li>OBART01G14130</li><li>KN539035.1_FG012</li><li>OGLUM01G15870</li><li>OMERI01G12860</li><li>OB01G23840</li><li>LPERR01G12840</li><li>BRADI_2g12520v3</li><li>SETIT_000849mg</li><li>SORBI_3003G155300</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os05g0156700</li><li>Os03g0163300</li><li>Os10g0415300</li><li>Os02g0813500</li><li>Os01g0328700</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>GSMUA_Achr1G18490_001</li><li>GSMUA_Achr3G12660_001</li><li>Dr16635</li><li>Dr02006</li><li>B456_006G259300</li><li>POPTR_010G142100v3</li><li>MANES_03G202500</li><li>GLYMA_15G143100</li><li>PHAVU_003G112300g</li><li>MANES_15G005700</li><li>HannXRQ_Chr05g0148211</li><li>LR48_Vigan635s008500</li><li>GLYMA_17G032300</li><li>POPTR_008G107600v3</li><li>PGSC0003DMG400027143</li><li>PGSC0003DMG400027215</li><li>LR48_Vigan01g300000</li><li>GLYMA_09G038000</li><li>GLYMA_07G241600</li><li>MTR_4g121880</li><li>PHAVU_009G254900g</li><li>B456_006G079100</li><li>MTR_2g034580</li><li>HannXRQ_Chr11g0339361</li><li>Bra012738</li><li>GSBRNA2T00031382001</li><li>Bo3g083510</li><li>GSBRNA2T00051655001</li><li>GSBRNA2T00154914001</li><li>fgenesh2_kg.3__1864__AT3G16950.2</li><li>AT4G16155</li><li>Bra001645</li><li>GSBRNA2T00151168001</li><li>GSBRNA2T00137509001</li><li>Bra033540</li><li>GSBRNA2T00056523001</li><li>Bo3g068570</li><li>Bo7g104250</li><li>AT3G16950</li><li>scaffold_702892.1</li><li>Bo1g055050</li><li>GSBRNA2T00123897001</li><li>SELMODRAFT_134914</li><li>SELMODRAFT_116004</li><li>PHYPA_011037</li><li>PHYPA_005407</li><li>PHYPA_016183</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0337900</td></tr><tr><th>Description</th><td>Similar to Dihydrolipoamide dehydrogenase. (Os01t0337900-01)</td></tr><tr><th>Position</th><td>chr01:13272614..13282008</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td></td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td></td></tr><tr><th>CGSNL Gene Symbol</th><td></td></tr><tr><th>CGSNL Gene Name</th><td></td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td></td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td></td></tr></table></td></tr></table>