<table style="width:100%" class="table table-striped" border="1"><tr><th>oryzabase</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>CGSNL Gene Symbol</th><td> ERF3 </td></tr><tr><th>Gene symbol synonym(s)</th><td> AP37 OsAP37 OsERF3 OsERF#075 OsERF075 OsERF75 ERF75 AP2/EREBP#004 AP2/EREBP4 OsBIERF2 BIERF2 </td></tr><tr><th>CGSNL Gene Name</th><td> ETHYLENE-RESPONSIVE ELEMENT-BINDING FACTOR 3 </td></tr><tr><th>Gene name synonym(s)</th><td> Apetela2 transcription factor 37 ethylene response factor 3 Ethylene responsive factor 3 ethylene response factor 75 APETALA2/ethylene-responsive element binding protein 4 benzothiadiazole (BTH)-induced ethylene responsive transcriptional factor 2 benzothiadiazole-induced ethylene responsive transcriptional factor 2 BTH-induced ethylene responsive transcriptional factor 2 ethylene-responsive element binding factor 3 </td></tr><tr><th>Chr. No.</th><td> 1 </td></tr><tr><th>Trait Class</th><td> Vegetative organ - Root Tolerance and resistance - Stress tolerance Character as QTL - Yield and productivity Other </td></tr><tr><th>Gene Ontology</th><td> GO:0010366 - negative regulation of ethylene biosynthetic process GO:0003700 - transcription factor activity GO:0003677 - DNA binding GO:0009414 - response to water deprivation GO:0009873 - ethylene mediated signaling pathway GO:0009733 - response to auxin stimulus GO:0009735 - response to cytokinin stimulus GO:0005634 - nucleus GO:0006351 - transcription, DNA-dependent GO:0009736 - cytokinin mediated signaling GO:0048364 - root development </td></tr><tr><th>Trait Ontology</th><td> TO:0000276 - drought tolerance TO:0000164 - stress trait TO:0000173 - ethylene sensitivity TO:0000656 - root development trait TO:0000167 - cytokinin sensitivity TO:0000163 - auxin sensitivity TO:0000371 - yield trait </td></tr><tr><th>Plant Ontology</th><td> PO:0000043 - crown root PO:0007518 - crown root emergence stage </td></tr><tr><th>RAP ID</th><td> Os01g0797600 Oryzabase ( IRGSP 1.0 / Build5 ) Rap ( IRGSP 1.0 / Build5 ) </td></tr><tr><th>Mutant Image</th><td> </td></tr></table></td></tr><tr><th>Gramene</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>_id</th><td>Os01g0797600</td></tr><tr><th>name</th><td>ETHYLENE-RESPONSIVE ELEMENT-BINDING FACTOR 3</td></tr><tr><th>description</th><td>AP2/ERF family protein, ERF-associated EAR-motif-containing repressor, Abiotic stress response, Stress signaling (Os01t0797600-01)</td></tr><tr><th>biotype</th><td>protein_coding</td></tr><tr><th>taxon_id</th><td>39947</td></tr><tr><th>system_name</th><td>oryza_sativa</td></tr><tr><th>db_type</th><td>core</td></tr><tr><th>gene_idx</th><td>8431</td></tr><tr><th>location</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>region</th><td>1</td></tr><tr><th>start</th><td>33766984</td></tr><tr><th>end</th><td>33768044</td></tr><tr><th>strand</th><td>1</td></tr><tr><th>map</th><td>GCA_001433935.1</td></tr></table></td></tr><tr><th>xrefs</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>db</th><th>ids</th></tr><tr><td>CGSNL</td><td><ul><li>ERF3</li></ul></td></tr><tr><td>UniParc</td><td><ul><li>UPI00000A1424</li></ul></td></tr><tr><td>Uniprot/SPTREMBL</td><td><ul><li>Q7F4G5</li></ul></td></tr><tr><td>EMBL</td><td><ul><li>AK061380</li></ul></td></tr><tr><td>protein_id</td><td><ul><li>BAS74765.1</li><li>BAB89900.1</li><li>EAZ13829.1</li><li>BAF06433.1</li></ul></td></tr><tr><td>RefSeq_peptide</td><td><ul><li>XP_015622082.1</li></ul></td></tr><tr><td>STRING</td><td><ul><li>39947.LOC_Os01g58420.1</li></ul></td></tr><tr><td>RefSeq_dna</td><td><ul><li>XM_015766596.1</li></ul></td></tr><tr><td>RAP-DB</td><td><ul><li>ERF3, AP37, OsAP37, OsERF3, OsERF#075, OsERF075, OsERF75, ERF75, AP2/EREBP#004, AP2/EREBP4, OsBIERF2, BIERF2</li></ul></td></tr><tr><td>Oryzabase</td><td><ul><li>AP37, OsAP37, OsERF3, OsERF#075, OsERF075, OsERF75, ERF75, AP2/EREBP#004, AP2/EREBP4, OsBIERF2, BIERF2</li></ul></td></tr><tr><td>EntrezGene</td><td><ul><li>4327621</li></ul></td></tr></table></td></tr><tr><th>annotations</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>domains</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>description</th></tr><tr><td>IPR001471</td><td>AP2/ERF_dom</td><td>AP2/ERF domain</td></tr><tr><td>IPR016177</td><td>DNA-bd_dom_sf</td><td>DNA-binding domain superfamily</td></tr><tr><td>IPR017392</td><td>AP2/ERF-transcript_factor</td><td>AP2/ERF transcription factor ERF/PTI6</td></tr><tr><td>IPR036955</td><td>AP2/ERF_dom_sf</td><td>AP2/ERF domain superfamily</td></tr></table></td></tr></table></td></tr><tr><th>GO</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>entries</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>id</th><th>name</th><th>namespace</th><th>def</th><th>evidence_code</th></tr><tr><td>GO:0010311</td><td>lateral root formation</td><td>biological_process</td><td>The process that gives rise to a lateral root. This process pertains to the initial formation of a structure from unspecified parts. A lateral root primordium represents an organized group of cells derived from the root pericycle that will differentiate into a new root, as opposed to the initiation of the main root from the embryo proper.</td><td>IMP</td></tr><tr><td>GO:0009751</td><td>response to salicylic acid</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a salicylic acid stimulus.</td><td>IEP</td></tr><tr><td>GO:0010728</td><td>regulation of hydrogen peroxide biosynthetic process</td><td>biological_process</td><td>Any process that modulates the rate, frequency or extent of hydrogen peroxide biosynthesis. The chemical reactions and pathways resulting in the formation of hydrogen peroxide (H2O2), a potentially harmful byproduct of aerobic cellular respiration which can cause damage to DNA.</td><td>IMP</td></tr><tr><td>GO:0010364</td><td>regulation of ethylene biosynthetic process</td><td>biological_process</td><td>Any process that modulates the frequency, rate, or extent of an ethylene biosynthetic process.</td><td>IMP</td></tr><tr><td>GO:0009737</td><td>response to abscisic acid</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an abscisic acid stimulus.</td><td>IEP</td></tr><tr><td>GO:0009611</td><td>response to wounding</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating damage to the organism.</td><td>IEP</td></tr><tr><td>GO:0009414</td><td>response to water deprivation</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a water deprivation stimulus, prolonged deprivation of water.</td><td>IMP</td></tr><tr><td>GO:0006351</td><td>transcription, DNA-templated</td><td>biological_process</td><td>The cellular synthesis of RNA on a template of DNA.</td><td>IEA</td></tr><tr><td>GO:0009735</td><td>response to cytokinin</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a cytokinin stimulus.</td><td>IEP</td></tr><tr><td>GO:0080141</td><td>regulation of jasmonic acid biosynthetic process</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of the chemical reactions and pathways resulting in the formation of jasmonic acid.</td><td>IMP</td></tr><tr><td>GO:0009409</td><td>response to cold</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a cold stimulus, a temperature stimulus below the optimal temperature for that organism.</td><td>IMP</td></tr><tr><td>GO:0045892</td><td>negative regulation of transcription, DNA-templated</td><td>biological_process</td><td>Any process that stops, prevents, or reduces the frequency, rate or extent of cellular DNA-templated transcription.</td><td>IDA</td></tr><tr><td>GO:0044212</td><td>transcription regulatory region DNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with a DNA region that regulates the transcription of a region of DNA, which may be a gene, cistron, or operon. Binding may occur as a sequence specific interaction or as an interaction observed only once a factor has been recruited to the DNA by other factors.</td><td>IDA</td></tr><tr><td>GO:2000068</td><td>regulation of defense response to insect</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of defense response to insect.</td><td>IMP</td></tr><tr><td>GO:0009625</td><td>response to insect</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus from an insect.</td><td>IEP</td></tr><tr><td>GO:0009723</td><td>response to ethylene</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an ethylene (ethene) stimulus.</td><td>IEP</td></tr><tr><td>GO:0043565</td><td>sequence-specific DNA binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with DNA of a specific nucleotide composition, e.g. GC-rich DNA binding, or with a specific sequence motif or type of DNA e.g. promotor binding or rDNA binding.</td><td>IDA</td></tr><tr><td>GO:0080027</td><td>response to herbivore</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus from a herbivore.</td><td>IEP</td></tr><tr><td>GO:0010337</td><td>regulation of salicylic acid metabolic process</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of the chemical reactions and pathways involving salicylic acid.</td><td>IMP</td></tr><tr><td>GO:0003700</td><td>DNA-binding transcription factor activity</td><td>molecular_function</td><td>A protein or a member of a complex that interacts selectively and non-covalently with a specific DNA sequence (sometimes referred to as a motif) within the regulatory region of a gene in order to modulate transcription. Regulatory regions include promoters (proximal and distal) and enhancers. Genes are transcriptional units, and include bacterial operons.</td><td>IDA</td></tr><tr><td>GO:0009753</td><td>response to jasmonic acid</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a jasmonic acid stimulus.</td><td>IEP</td></tr><tr><td>GO:0009651</td><td>response to salt stress</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a stimulus indicating an increase or decrease in the concentration of salt (particularly but not exclusively sodium and chloride ions) in the environment.</td><td>IMP</td></tr><tr><td>GO:0009733</td><td>response to auxin</td><td>biological_process</td><td>Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of an auxin stimulus.</td><td>IEP</td></tr><tr><td>GO:0005634</td><td>nucleus</td><td>cellular_component</td><td>A membrane-bounded organelle of eukaryotic cells in which chromosomes are housed and replicated. In most cells, the nucleus contains all of the cell's chromosomes except the organellar chromosomes, and is the site of RNA synthesis and processing. In some species, or in specialized cell types, RNA metabolism or DNA replication may be absent.</td><td>IDA</td></tr><tr><td>GO:0080113</td><td>regulation of seed growth</td><td>biological_process</td><td>Any process that modulates the frequency, rate or extent of growth of the seed of an plant.</td><td>IMP</td></tr><tr><td>GO:0005515</td><td>protein binding</td><td>molecular_function</td><td>Interacting selectively and non-covalently with any protein or protein complex (a complex of two or more proteins that may include other nonprotein molecules).</td><td>IPI</td></tr></table></td></tr></table></td></tr></table></td></tr><tr><th>homology</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>homologous_genes</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ortholog_one2one</th><td><ul><li>ORUFI01G37120</li><li>BGIOSGA004611</li><li>ORGLA01G0278100</li><li>OBART01G33870</li><li>OGLUM01G38110</li><li>KN539005.1_FG012</li><li>OMERI01G30710</li><li>OPUNC01G32930</li><li>OB01G42720</li><li>LPERR01G29260</li><li>TraesCS3B01G357500</li><li>TraesCS3A01G328000</li><li>TraesCS3D01G321500</li><li>BRADI_2g52370v3</li><li>SORBI_3003G324400</li><li>SETIT_002729mg</li></ul></td></tr><tr><th>within_species_paralog</th><td><ul><li>Os05g0497300</li><li>Os04g0610400</li><li>Os05g0497200</li><li>Os06g0691100</li><li>Os04g0669200</li><li>Os04g0399800</li><li>Os02g0520000</li><li>Os08g0173700</li><li>Os07g0669500</li><li>Os02g0594300</li></ul></td></tr><tr><th>ortholog_many2many</th><td><ul><li>GSMUA_Achr10G09150_001</li><li>GSMUA_Achr8G03550_001</li><li>Tp57577_TGAC_v2_gene11640</li><li>HannXRQ_Chr04g0123861</li><li>POPTR_011G057000v3</li><li>Csa_7G049230</li><li>MTR_4g078710</li><li>MANES_16G030600</li><li>GLYMA_13G236500</li><li>B456_010G155900</li><li>B456_011G029800</li><li>B456_006G088000</li><li>Csa_6G518040</li><li>Csa_1G075060</li><li>GLYMA_15G077100</li><li>B456_005G200200</li><li>POPTR_004G047600v3</li><li>HannXRQ_Chr16g0518421</li><li>MTR_2g014340</li><li>B456_010G256600</li><li>GLYMA_08G278800</li><li>GLYMA_10G061400</li><li>Tp57577_TGAC_v2_gene35959</li><li>HannXRQ_Chr16g0517611</li><li>HannXRQ_Chr17g0561861</li><li>MANES_17G048400</li><li>PGSC0003DMG400022823</li><li>PGSC0003DMG400013744</li><li>Solyc03g006320.1</li><li>Solyc07g053740.1</li><li>PGSC0003DMG400026821</li><li>GSBRNA2T00150330001</li><li>Solyc10g009110.1</li><li>PGSC0003DMG400012828</li><li>GSBRNA2T00002713001</li><li>SELMODRAFT_72880</li><li>SELMODRAFT_73144</li><li>PHYPA_021885</li><li>PHYPA_020435</li><li>PHYPA_030745</li><li>PHYPA_030111</li><li>PHYPA_019186</li><li>PHYPA_012149</li></ul></td></tr></table></td></tr></table></td></tr><tr><th>species_idx</th><td>5</td></tr></table></td></tr><tr><th>funricegene_genekeywords</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'AP37|OsERF3'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0797600'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g58420'}</td></tr><tr><th>Keyword</th><td>{'grain', 'transcription factor', 'insect', 'grain yield', 'drought tolerance', 'root initiation', 'crown root development', 'drought', 'defense response', 'root', 'root development', 'stem', 'crown root elongation', 'yield', 'crown root initiation', 'temperature', 'salinity', 'root elongation', 'ethylene', 'jasmonate', 'vegetative', 'brown planthopper', 'cytokinin', ' ABA ', 'auxin', 'defense', 'growth', 'salt'}</td></tr><tr><th>Title</th><td>{'Transcriptional activation of OsDERF1 in OsERF3 and OsAP2-39 negatively modulates ethylene synthesis and drought tolerance in rice', 'An EAR-motif-containing ERF transcription factor affects herbivore-induced signaling, defense and resistance in rice', 'The Interaction between Rice ERF3 and WOX11 Promotes Crown Root Development by Regulating Gene Expression Involved in Cytokinin Signaling', 'Overexpression of the Transcription Factor AP37 in Rice Improves Grain Yield under Drought Conditions', 'Isolation and identification of an AP2/ERF factor that binds an allelic cis-element of rice gene LRK6', 'EAR motif mutation of rice OsERF3 alters the regulation of ethylene biosynthesis and drought tolerance'}</td></tr></table></td></tr><tr><th>rapdb</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>ID</th><td>Os01g0797600</td></tr><tr><th>Description</th><td>AP2/ERF family protein, ERF-associated EAR-motif-containing repressor, Abiotic stress response, Stress signaling (Os01t0797600-01)</td></tr><tr><th>Position</th><td>chr01:33766984..33768044</td></tr><tr><th>RAP-DB Gene Symbol Synonym(s)</th><td>ERF3, AP37, OsAP37, OsERF3, OsERF#075, OsERF075, OsERF75, ERF75, AP2/EREBP#004, AP2/EREBP4, OsBIERF2, BIERF2</td></tr><tr><th>RAP-DB Gene Name Synonym(s)</th><td>ETHYLENE-RESPONSIVE ELEMENT-BINDING FACTOR 3, Apetela2 transcription factor 37, ethylene response factor 3, Ethylene responsive factor 3, ethylene response factor 75, APETALA2/ethylene-responsive element binding protein 4, benzothiadiazole (BTH)-induced ethylene responsive transcriptional factor 2, benzothiadiazole-induced ethylene responsive transcriptional factor 2, BTH-induced ethylene responsive transcriptional factor 2, ethylene-responsive element binding factor 3</td></tr><tr><th>CGSNL Gene Symbol</th><td>ERF3</td></tr><tr><th>CGSNL Gene Name</th><td>ETHYLENE-RESPONSIVE ELEMENT-BINDING FACTOR 3</td></tr><tr><th>Oryzabase Gene Symbol Synonym(s)</th><td>AP37, OsAP37, OsERF3, OsERF#075, OsERF075, OsERF75, ERF75, AP2/EREBP#004, AP2/EREBP4, OsBIERF2, BIERF2</td></tr><tr><th>Oryzabase Gene Name Synonym(s)</th><td>Apetela2 transcription factor 37, ethylene response factor 3, Ethylene responsive factor 3, ethylene response factor 75, APETALA2/ethylene-responsive element binding protein 4, benzothiadiazole (BTH)-induced ethylene responsive transcriptional factor 2, benzothiadiazole-induced ethylene responsive transcriptional factor 2, BTH-induced ethylene responsive transcriptional factor 2, ethylene-responsive element binding factor 3</td></tr></table></td></tr><tr><th>funricegene_geneinfo</th><td><table style="width:100%" class="table table-striped" border="1"><tr><th>Symbol</th><td>{'AP37|OsERF3'}</td></tr><tr><th>RAPdb</th><td>{'Os01g0797600'}</td></tr><tr><th>MSU</th><td>{'LOC_Os01g58420'}</td></tr></table></td></tr></table>